<!DOCTYPE html>

<html lang="en">
<head>
<meta content="text/html; charset=utf-8" http-equiv="content-type"/>
<title>MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion</title>
<!--Generated on Mon Feb 19 02:30:53 2024 by LaTeXML (version 0.8.7) http://dlmf.nist.gov/LaTeXML/.-->
<meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport"/>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv_0.7.4.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/latexml_styles.css" rel="stylesheet" type="text/css"/>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.3.3/html2canvas.min.js"></script>
<script src="/static/browse/0.3.4/js/addons.js"></script>
<script src="/static/browse/0.3.4/js/feedbackOverlay.js"></script>
<base href="/html/2402.11788v1/"/></head>
<body>
<nav class="ltx_page_navbar">
<nav class="ltx_TOC">
<ol class="ltx_toclist">
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S1" title="1 Introduction â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1 </span>Introduction</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S2" title="2 Related Works â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2 </span>Related Works</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S3" title="3 Data Preparation â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3 </span>Data Preparation</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4" title="4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4 </span>Proposed Method</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.SS1" title="4.1 Histopathological Feature Extraction â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.1 </span>Histopathological Feature Extraction</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.SS2" title="4.2 Patch-to-Patient Aggregation Using Self-Attention â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.2 </span>Patch-to-Patient Aggregation Using Self-Attention</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.SS3" title="4.3 Multimodal Fusion Using Dual Cross-Attention â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.3 </span>Multimodal Fusion Using Dual Cross-Attention</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.SS4" title="4.4 MM-SurvNet Model Training â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.4 </span>MM-SurvNet Model Training</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S5" title="5 Experimental Results â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5 </span>Experimental Results</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S6" title="6 Discussion â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6 </span>Discussion</span></a></li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S7" title="7 Conclusion â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">7 </span>Conclusion</span></a></li>
</ol></nav>
</nav>
<div class="ltx_page_main">
<div class="ltx_page_content">
<div aria-label="Conversion errors have been found" class="package-alerts ltx_document" role="status">
<button aria-label="Dismiss alert" onclick="closePopup()">
<span aria-hidden="true"><svg aria-hidden="true" focusable="false" height="20" role="presentation" viewbox="0 0 44 44" width="20">
<path d="M0.549989 4.44999L4.44999 0.549988L43.45 39.55L39.55 43.45L0.549989 4.44999Z"></path>
<path d="M39.55 0.549988L43.45 4.44999L4.44999 43.45L0.549988 39.55L39.55 0.549988Z"></path>
</svg></span>
</button>
<p>HTML conversions <a href="https://info.dev.arxiv.org/about/accessibility_html_error_messages.html" target="_blank">sometimes display errors</a> due to content that did not convert correctly from the source. This paper uses the following packages that are not yet supported by the HTML conversion tool. Feedback on these issues are not necessary; they are known and are being worked on.</p>
<ul arial-label="Unsupported packages used in this paper">
<li>failed: academicons</li>
</ul>
<p>Authors: achieve the best HTML results from your LaTeX submissions by following these <a href="https://info.arxiv.org/help/submit_latex_best_practices.html" target="_blank">best practices</a>.</p>
</div><div class="section" id="target-section"><div id="license-tr">License: CC BY 4.0</div><div id="watermark-tr">arXiv:2402.11788v1 [cs.CV] 19 Feb 2024</div></div>
<script>
            function closePopup() {
                document.querySelector('.package-alerts').style.display = 'none';
            }
        </script>
<article class="ltx_document ltx_authors_1line"><span class="ltx_ERROR undefined" id="id1">\sidecaptionvpos</span>
<div class="ltx_para" id="p1">
<p class="ltx_p" id="p1.1">figurec







































</p>
</div>
<div class="ltx_para ltx_noindent" id="p2">
</div>
<h1 class="ltx_title ltx_title_document">MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Raktim Kumar Mondol
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">School of Computer Science and Engineering, University of New South Wales, Sydney, Australia
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Ewan K.A. Millar
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">Department of Anatomical Pathology, NSW Health Pathology, St. George Hospital
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Arcot Sowmya
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">School of Computer Science and Engineering, University of New South Wales, Sydney, Australia
</span></span></span>
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Erik Meijering
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_affiliation">School of Computer Science and Engineering, University of New South Wales, Sydney, Australia
</span>
<span class="ltx_contact ltx_role_affiliation">Correspondence: <span class="ltx_text ltx_font_typewriter" id="id1.1.id1">erik.meijering@unsw.edu.au</span>
</span></span></span>
</div>
<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p class="ltx_p" id="id2.id1">Survival risk stratification is an important step in clinical decision making for breast cancer management. We propose a novel deep learning approach for this purpose by integrating histopathological imaging, genetic and clinical data. It employs vision transformers, specifically the MaxViT model, for image feature extraction, and self-attention to capture intricate image relationships at the patient level. A dual cross-attention mechanism fuses these features with genetic data, while clinical data is incorporated at the final layer to enhance predictive accuracy. Experiments on the public TCGA-BRCA dataset show that our model, trained using the negative log likelihood loss function, can achieve superior performance with a mean C-index of 0.64, surpassing existing methods. This advancement facilitates tailored treatment strategies, potentially leading to improved patient outcomes.</p>
</div>
<div class="ltx_para ltx_noindent" id="p3">
<p class="ltx_p" id="p3.4"><em class="ltx_emph ltx_font_bold ltx_font_italic" id="p3.4.1">K</em><span class="ltx_text ltx_font_bold" id="p3.4.2">eywords</span>â€‚Multimodal FusionÂ <math alttext="\cdot" class="ltx_Math" display="inline" id="p3.1.m1.1"><semantics id="p3.1.m1.1a"><mo id="p3.1.m1.1.1" xref="p3.1.m1.1.1.cmml">â‹…</mo><annotation-xml encoding="MathML-Content" id="p3.1.m1.1b"><ci id="p3.1.m1.1.1.cmml" xref="p3.1.m1.1.1">â‹…</ci></annotation-xml><annotation encoding="application/x-tex" id="p3.1.m1.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p3.1.m1.1d">â‹…</annotation></semantics></math>
Breast CancerÂ <math alttext="\cdot" class="ltx_Math" display="inline" id="p3.2.m2.1"><semantics id="p3.2.m2.1a"><mo id="p3.2.m2.1.1" xref="p3.2.m2.1.1.cmml">â‹…</mo><annotation-xml encoding="MathML-Content" id="p3.2.m2.1b"><ci id="p3.2.m2.1.1.cmml" xref="p3.2.m2.1.1">â‹…</ci></annotation-xml><annotation encoding="application/x-tex" id="p3.2.m2.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p3.2.m2.1d">â‹…</annotation></semantics></math>
Whole Slide ImagesÂ <math alttext="\cdot" class="ltx_Math" display="inline" id="p3.3.m3.1"><semantics id="p3.3.m3.1a"><mo id="p3.3.m3.1.1" xref="p3.3.m3.1.1.cmml">â‹…</mo><annotation-xml encoding="MathML-Content" id="p3.3.m3.1b"><ci id="p3.3.m3.1.1.cmml" xref="p3.3.m3.1.1">â‹…</ci></annotation-xml><annotation encoding="application/x-tex" id="p3.3.m3.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p3.3.m3.1d">â‹…</annotation></semantics></math>
Deep Neural NetworkÂ <math alttext="\cdot" class="ltx_Math" display="inline" id="p3.4.m4.1"><semantics id="p3.4.m4.1a"><mo id="p3.4.m4.1.1" xref="p3.4.m4.1.1.cmml">â‹…</mo><annotation-xml encoding="MathML-Content" id="p3.4.m4.1b"><ci id="p3.4.m4.1.1.cmml" xref="p3.4.m4.1.1">â‹…</ci></annotation-xml><annotation encoding="application/x-tex" id="p3.4.m4.1c">\cdot</annotation><annotation encoding="application/x-llamapun" id="p3.4.m4.1d">â‹…</annotation></semantics></math>
Survival Prediction</p>
</div>
<section class="ltx_section" id="S1">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>
<div class="ltx_para ltx_noindent" id="S1.p1">
<p class="ltx_p" id="S1.p1.1">The inherent heterogeneity of breast cancer poses challenges for prediction of prognosis and treatment decisions particularly in post-menopausal estrogen receptor positive (ER+) breast cancer, with previous studies reporting conflicting results on the survival difference between Luminal A and B metastatic breast cancer patientsÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib1" title="">1</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib2" title="">2</a>]</cite>. A common critical clinical dilemma is the selection of those early ER+ breast cancer patients at high risk of recurrence who may benefit from the addition of chemotherapy to endocrine therapy. Traditional risk prediction methods, based mainly on clinicopathological factors, may not entirely account for the complex biology of cancerÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib3" title="">3</a>]</cite>. To address this issue, molecular markers and gene expression profiles have emerged as a potential addition to current decision making toolsÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib4" title="">4</a>]</cite>. Therefore, combining histopathology image, molecular and clinicopathological factors (see Figure <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S1.F1" title="Figure 1 â€£ 1 Introduction â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">1</span></a> adapted from <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib5" title="">5</a>]</cite>) into a single risk prediction model could potentially enhance risk prognostication in the clinicÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib6" title="">6</a>]</cite>. Here we propose a multimodal risk prediction model for ER+ breast cancer and compare its performance to existing methods. The significance of our work lies in potentially refining ER+ breast cancer survival risk prediction, facilitating informed therapeutic decision makingÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib7" title="">7</a>]</cite>.</p>
</div>
<figure class="ltx_figure" id="S1.F1"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="197" id="S1.F1.g1" src="x1.png" width="406"/>
<br class="ltx_break ltx_centering"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>Multimodal Data for Breast Cancer Characterization: This figure illustrates the diverse data types at multiple biological levels to comprehensively characterize breast cancer. It encompasses clinical data at the patient level, mammogram images at the organ level, histopathology images at the tissue level, and gene expression data (e.g., RNA Sequencing) at the molecular level, thereby providing a holistic view of the cancerâ€™s nature and behavior. </figcaption>
</figure>
<figure class="ltx_figure" id="S1.F2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="282" id="S1.F2.g1" src="x2.png" width="789"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Preprocessing Methodology for Histopathology Images: Whole Slide Images (WSIs) are first annotated using the QuPath annotation tool by expert pathologists. From these annotated regions, non-overlapping patches are systematically extracted. Subsequently, a color transformation is applied to ensure uniformity in color representation across datasets.</figcaption>
</figure>
</section>
<section class="ltx_section" id="S2">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Related Works</h2>
<div class="ltx_para ltx_noindent" id="S2.p1">
<p class="ltx_p" id="S2.p1.1">Breast cancer accounts for about 30% of all cancer cases in women, with varying incidence rates across regions and notably higher rates in developed countriesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib8" title="">8</a>]</cite>. The ER status, specifically ER+ breast cancer, which encompasses Luminal A and Luminal B subtypes, is crucial in shaping treatment strategies and predicting prognosisÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib3" title="">3</a>]</cite>. While Luminal A tumours are usually low-grade with a favorable prognosis (Ki-67 &lt; 14%), Luminal B tumours are usually higher grade and pose a higher recurrence risk and worse outcome (Ki-67 <math alttext="\geq" class="ltx_Math" display="inline" id="S2.p1.1.m1.1"><semantics id="S2.p1.1.m1.1a"><mo id="S2.p1.1.m1.1.1" xref="S2.p1.1.m1.1.1.cmml">â‰¥</mo><annotation-xml encoding="MathML-Content" id="S2.p1.1.m1.1b"><geq id="S2.p1.1.m1.1.1.cmml" xref="S2.p1.1.m1.1.1"></geq></annotation-xml><annotation encoding="application/x-tex" id="S2.p1.1.m1.1c">\geq</annotation><annotation encoding="application/x-llamapun" id="S2.p1.1.m1.1d">â‰¥</annotation></semantics></math> 14%)Â <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib3" title="">3</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib9" title="">9</a>]</cite>. Traditional prediction models primarily centred on clinical and pathological factors fall short, especially in capturing the intricacies and heterogeneity of ER+ breast cancer subtypesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib10" title="">10</a>]</cite>. Transitioning from traditional methods, early research in survival risk prediction using histopathology images shifted from reliance on hand-crafted features like texture and shape to leveraging deep learning for automatic extraction of complex, high-level features, resulting in enhanced accuracyÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib11" title="">11</a>]</cite>. In recent developments, multimodal approaches that combine imaging, clinical and molecular data have been shown to enhance prediction accuracyÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib12" title="">12</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib13" title="">13</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib14" title="">14</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib15" title="">15</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib16" title="">16</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib17" title="">17</a>]</cite>. Comprehensive multimodal approaches, especially those that fuse whole slide images, clinical data and genetic information, could pave the way for superior predictive models.</p>
</div>
</section>
<section class="ltx_section" id="S3">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Data Preparation</h2>
<div class="ltx_para" id="S3.p1">
<p class="ltx_p" id="S3.p1.1">Our study uses the hematoxylin and eosin (H&amp;E)-stained formalin-fixed paraffin-embedded (FFPE) digital slides of The Cancer Genome Atlas Breast Cancer (TCGA-BRCA) dataset <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib18" title="">18</a>]</cite>, sourcing 249 whole-slide images (WSIs) from the Genomic Data Commons (GDC) Portal with a focus on Luminal A (149 samples) and Luminal B (100 samples) molecular subtypes. Slide annotations were obtained manually by an expert breast pathologist using QuPath <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib19" title="">19</a>]</cite>, prioritising tumour localisation including stroma and tumour-infiltrating lymphocytes (TILs) and excluding necrotic areas. H&amp;E-stained tissues underwent downsampling to a resolution of 0.25 Âµm/pixel, with tissue masks created to exclude artifacts and non-tissue sections, and tumour regions split into <math alttext="224\times 224" class="ltx_Math" display="inline" id="S3.p1.1.m1.1"><semantics id="S3.p1.1.m1.1a"><mrow id="S3.p1.1.m1.1.1" xref="S3.p1.1.m1.1.1.cmml"><mn id="S3.p1.1.m1.1.1.2" xref="S3.p1.1.m1.1.1.2.cmml">224</mn><mo id="S3.p1.1.m1.1.1.1" lspace="0.222em" rspace="0.222em" xref="S3.p1.1.m1.1.1.1.cmml">Ã—</mo><mn id="S3.p1.1.m1.1.1.3" xref="S3.p1.1.m1.1.1.3.cmml">224</mn></mrow><annotation-xml encoding="MathML-Content" id="S3.p1.1.m1.1b"><apply id="S3.p1.1.m1.1.1.cmml" xref="S3.p1.1.m1.1.1"><times id="S3.p1.1.m1.1.1.1.cmml" xref="S3.p1.1.m1.1.1.1"></times><cn id="S3.p1.1.m1.1.1.2.cmml" type="integer" xref="S3.p1.1.m1.1.1.2">224</cn><cn id="S3.p1.1.m1.1.1.3.cmml" type="integer" xref="S3.p1.1.m1.1.1.3">224</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S3.p1.1.m1.1c">224\times 224</annotation><annotation encoding="application/x-llamapun" id="S3.p1.1.m1.1d">224 Ã— 224</annotation></semantics></math> pixel patches. To counteract staining inconsistencies, singular value decomposition-based normalisation was applied on the images (See FigureÂ <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S1.F2" title="Figure 2 â€£ 1 Introduction â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">2</span></a>) using Macenko method <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib20" title="">20</a>]</cite>. Furthermore, we processed RNA-sequencing data with RSEM and selected PAM50 genes for analysis, given their significance in breast cancer subtypingÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib21" title="">21</a>, <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib22" title="">22</a>]</cite>. Pertinent clinical variables, encompassing tumour grade, size, patient age and lymph node status, were meticulously prepared to enhance the predictive accuracy of the subsequent deep learning model evaluations. For experimental purposes, we adopted a five-fold cross-validation strategy, allocating 200 samples for training and 50 for validation while maintaining the distribution of Luminal subtypes and survival statuses.</p>
</div>
<figure class="ltx_figure" id="S3.F3"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="397" id="S3.F3.g1" src="x3.png" width="748"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>The proposed MM-SurvNet architecture. It utilises a pretrained MaxViT to extract features from histopathology images, which are then processed through a self-attention network to aggregate patch-level features into a patient-level representation.</figcaption>
</figure>
</section>
<section class="ltx_section" id="S4">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Proposed Method</h2>
<section class="ltx_subsection" id="S4.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Histopathological Feature Extraction</h3>
<div class="ltx_para ltx_noindent" id="S4.SS1.p1">
<p class="ltx_p" id="S4.SS1.p1.1">We utilised vision transformers (ViTs) and convolutional neural networks (CNNs) to extract features from histopathological images. Specifically, the MaxViT model, a prominent vision transformer, was employed for its effectiveness in capturing global image information and leveraging self-attention for comprehensive tumour analysisÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib23" title="">23</a>]</cite>. For the purpose of comparison, ResNet50, a CNN-based model, was selected to extract local patterns within the imagesÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib24" title="">24</a>]</cite>.</p>
</div>
</section>
<section class="ltx_subsection" id="S4.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Patch-to-Patient Aggregation Using Self-Attention</h3>
<div class="ltx_para ltx_noindent" id="S4.SS2.p1">
<p class="ltx_p" id="S4.SS2.p1.1">In our framework, the self-attention mechanism crucially aggregates patch-level features into a holistic patient-level representation. Notably, the same image patches act as the Key (K), Query (Q), and Value (V) within the attention paradigm, aiming to discern intricate interdependencies among patches from a singular histopathological image.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS2.p2">
<p class="ltx_p" id="S4.SS2.p2.4">Mathematically, self-attention processes a sequence of image patches <math alttext="\mathbf{K}_{1},\mathbf{Q}_{1},\mathbf{V}_{1},\ldots,\mathbf{K}_{n},\mathbf{Q}_%
{n},\mathbf{V}_{n}" class="ltx_Math" display="inline" id="S4.SS2.p2.1.m1.7"><semantics id="S4.SS2.p2.1.m1.7a"><mrow id="S4.SS2.p2.1.m1.7.7.6" xref="S4.SS2.p2.1.m1.7.7.7.cmml"><msub id="S4.SS2.p2.1.m1.2.2.1.1" xref="S4.SS2.p2.1.m1.2.2.1.1.cmml"><mi id="S4.SS2.p2.1.m1.2.2.1.1.2" xref="S4.SS2.p2.1.m1.2.2.1.1.2.cmml">ğŠ</mi><mn id="S4.SS2.p2.1.m1.2.2.1.1.3" xref="S4.SS2.p2.1.m1.2.2.1.1.3.cmml">1</mn></msub><mo id="S4.SS2.p2.1.m1.7.7.6.7" xref="S4.SS2.p2.1.m1.7.7.7.cmml">,</mo><msub id="S4.SS2.p2.1.m1.3.3.2.2" xref="S4.SS2.p2.1.m1.3.3.2.2.cmml"><mi id="S4.SS2.p2.1.m1.3.3.2.2.2" xref="S4.SS2.p2.1.m1.3.3.2.2.2.cmml">ğ</mi><mn id="S4.SS2.p2.1.m1.3.3.2.2.3" xref="S4.SS2.p2.1.m1.3.3.2.2.3.cmml">1</mn></msub><mo id="S4.SS2.p2.1.m1.7.7.6.8" xref="S4.SS2.p2.1.m1.7.7.7.cmml">,</mo><msub id="S4.SS2.p2.1.m1.4.4.3.3" xref="S4.SS2.p2.1.m1.4.4.3.3.cmml"><mi id="S4.SS2.p2.1.m1.4.4.3.3.2" xref="S4.SS2.p2.1.m1.4.4.3.3.2.cmml">ğ•</mi><mn id="S4.SS2.p2.1.m1.4.4.3.3.3" xref="S4.SS2.p2.1.m1.4.4.3.3.3.cmml">1</mn></msub><mo id="S4.SS2.p2.1.m1.7.7.6.9" xref="S4.SS2.p2.1.m1.7.7.7.cmml">,</mo><mi id="S4.SS2.p2.1.m1.1.1" mathvariant="normal" xref="S4.SS2.p2.1.m1.1.1.cmml">â€¦</mi><mo id="S4.SS2.p2.1.m1.7.7.6.10" xref="S4.SS2.p2.1.m1.7.7.7.cmml">,</mo><msub id="S4.SS2.p2.1.m1.5.5.4.4" xref="S4.SS2.p2.1.m1.5.5.4.4.cmml"><mi id="S4.SS2.p2.1.m1.5.5.4.4.2" xref="S4.SS2.p2.1.m1.5.5.4.4.2.cmml">ğŠ</mi><mi id="S4.SS2.p2.1.m1.5.5.4.4.3" xref="S4.SS2.p2.1.m1.5.5.4.4.3.cmml">n</mi></msub><mo id="S4.SS2.p2.1.m1.7.7.6.11" xref="S4.SS2.p2.1.m1.7.7.7.cmml">,</mo><msub id="S4.SS2.p2.1.m1.6.6.5.5" xref="S4.SS2.p2.1.m1.6.6.5.5.cmml"><mi id="S4.SS2.p2.1.m1.6.6.5.5.2" xref="S4.SS2.p2.1.m1.6.6.5.5.2.cmml">ğ</mi><mi id="S4.SS2.p2.1.m1.6.6.5.5.3" xref="S4.SS2.p2.1.m1.6.6.5.5.3.cmml">n</mi></msub><mo id="S4.SS2.p2.1.m1.7.7.6.12" xref="S4.SS2.p2.1.m1.7.7.7.cmml">,</mo><msub id="S4.SS2.p2.1.m1.7.7.6.6" xref="S4.SS2.p2.1.m1.7.7.6.6.cmml"><mi id="S4.SS2.p2.1.m1.7.7.6.6.2" xref="S4.SS2.p2.1.m1.7.7.6.6.2.cmml">ğ•</mi><mi id="S4.SS2.p2.1.m1.7.7.6.6.3" xref="S4.SS2.p2.1.m1.7.7.6.6.3.cmml">n</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="S4.SS2.p2.1.m1.7b"><list id="S4.SS2.p2.1.m1.7.7.7.cmml" xref="S4.SS2.p2.1.m1.7.7.6"><apply id="S4.SS2.p2.1.m1.2.2.1.1.cmml" xref="S4.SS2.p2.1.m1.2.2.1.1"><csymbol cd="ambiguous" id="S4.SS2.p2.1.m1.2.2.1.1.1.cmml" xref="S4.SS2.p2.1.m1.2.2.1.1">subscript</csymbol><ci id="S4.SS2.p2.1.m1.2.2.1.1.2.cmml" xref="S4.SS2.p2.1.m1.2.2.1.1.2">ğŠ</ci><cn id="S4.SS2.p2.1.m1.2.2.1.1.3.cmml" type="integer" xref="S4.SS2.p2.1.m1.2.2.1.1.3">1</cn></apply><apply id="S4.SS2.p2.1.m1.3.3.2.2.cmml" xref="S4.SS2.p2.1.m1.3.3.2.2"><csymbol cd="ambiguous" id="S4.SS2.p2.1.m1.3.3.2.2.1.cmml" xref="S4.SS2.p2.1.m1.3.3.2.2">subscript</csymbol><ci id="S4.SS2.p2.1.m1.3.3.2.2.2.cmml" xref="S4.SS2.p2.1.m1.3.3.2.2.2">ğ</ci><cn id="S4.SS2.p2.1.m1.3.3.2.2.3.cmml" type="integer" xref="S4.SS2.p2.1.m1.3.3.2.2.3">1</cn></apply><apply id="S4.SS2.p2.1.m1.4.4.3.3.cmml" xref="S4.SS2.p2.1.m1.4.4.3.3"><csymbol cd="ambiguous" id="S4.SS2.p2.1.m1.4.4.3.3.1.cmml" xref="S4.SS2.p2.1.m1.4.4.3.3">subscript</csymbol><ci id="S4.SS2.p2.1.m1.4.4.3.3.2.cmml" xref="S4.SS2.p2.1.m1.4.4.3.3.2">ğ•</ci><cn id="S4.SS2.p2.1.m1.4.4.3.3.3.cmml" type="integer" xref="S4.SS2.p2.1.m1.4.4.3.3.3">1</cn></apply><ci id="S4.SS2.p2.1.m1.1.1.cmml" xref="S4.SS2.p2.1.m1.1.1">â€¦</ci><apply id="S4.SS2.p2.1.m1.5.5.4.4.cmml" xref="S4.SS2.p2.1.m1.5.5.4.4"><csymbol cd="ambiguous" id="S4.SS2.p2.1.m1.5.5.4.4.1.cmml" xref="S4.SS2.p2.1.m1.5.5.4.4">subscript</csymbol><ci id="S4.SS2.p2.1.m1.5.5.4.4.2.cmml" xref="S4.SS2.p2.1.m1.5.5.4.4.2">ğŠ</ci><ci id="S4.SS2.p2.1.m1.5.5.4.4.3.cmml" xref="S4.SS2.p2.1.m1.5.5.4.4.3">ğ‘›</ci></apply><apply id="S4.SS2.p2.1.m1.6.6.5.5.cmml" xref="S4.SS2.p2.1.m1.6.6.5.5"><csymbol cd="ambiguous" id="S4.SS2.p2.1.m1.6.6.5.5.1.cmml" xref="S4.SS2.p2.1.m1.6.6.5.5">subscript</csymbol><ci id="S4.SS2.p2.1.m1.6.6.5.5.2.cmml" xref="S4.SS2.p2.1.m1.6.6.5.5.2">ğ</ci><ci id="S4.SS2.p2.1.m1.6.6.5.5.3.cmml" xref="S4.SS2.p2.1.m1.6.6.5.5.3">ğ‘›</ci></apply><apply id="S4.SS2.p2.1.m1.7.7.6.6.cmml" xref="S4.SS2.p2.1.m1.7.7.6.6"><csymbol cd="ambiguous" id="S4.SS2.p2.1.m1.7.7.6.6.1.cmml" xref="S4.SS2.p2.1.m1.7.7.6.6">subscript</csymbol><ci id="S4.SS2.p2.1.m1.7.7.6.6.2.cmml" xref="S4.SS2.p2.1.m1.7.7.6.6.2">ğ•</ci><ci id="S4.SS2.p2.1.m1.7.7.6.6.3.cmml" xref="S4.SS2.p2.1.m1.7.7.6.6.3">ğ‘›</ci></apply></list></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p2.1.m1.7c">\mathbf{K}_{1},\mathbf{Q}_{1},\mathbf{V}_{1},\ldots,\mathbf{K}_{n},\mathbf{Q}_%
{n},\mathbf{V}_{n}</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p2.1.m1.7d">bold_K start_POSTSUBSCRIPT 1 end_POSTSUBSCRIPT , bold_Q start_POSTSUBSCRIPT 1 end_POSTSUBSCRIPT , bold_V start_POSTSUBSCRIPT 1 end_POSTSUBSCRIPT , â€¦ , bold_K start_POSTSUBSCRIPT italic_n end_POSTSUBSCRIPT , bold_Q start_POSTSUBSCRIPT italic_n end_POSTSUBSCRIPT , bold_V start_POSTSUBSCRIPT italic_n end_POSTSUBSCRIPT</annotation></semantics></math> to yield a patient-level feature vector <math alttext="\mathbf{y}" class="ltx_Math" display="inline" id="S4.SS2.p2.2.m2.1"><semantics id="S4.SS2.p2.2.m2.1a"><mi id="S4.SS2.p2.2.m2.1.1" xref="S4.SS2.p2.2.m2.1.1.cmml">ğ²</mi><annotation-xml encoding="MathML-Content" id="S4.SS2.p2.2.m2.1b"><ci id="S4.SS2.p2.2.m2.1.1.cmml" xref="S4.SS2.p2.2.m2.1.1">ğ²</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p2.2.m2.1c">\mathbf{y}</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p2.2.m2.1d">bold_y</annotation></semantics></math>. The mechanism computes a weighted sum of the Value vectors <math alttext="\mathbf{V}_{j}" class="ltx_Math" display="inline" id="S4.SS2.p2.3.m3.1"><semantics id="S4.SS2.p2.3.m3.1a"><msub id="S4.SS2.p2.3.m3.1.1" xref="S4.SS2.p2.3.m3.1.1.cmml"><mi id="S4.SS2.p2.3.m3.1.1.2" xref="S4.SS2.p2.3.m3.1.1.2.cmml">ğ•</mi><mi id="S4.SS2.p2.3.m3.1.1.3" xref="S4.SS2.p2.3.m3.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p2.3.m3.1b"><apply id="S4.SS2.p2.3.m3.1.1.cmml" xref="S4.SS2.p2.3.m3.1.1"><csymbol cd="ambiguous" id="S4.SS2.p2.3.m3.1.1.1.cmml" xref="S4.SS2.p2.3.m3.1.1">subscript</csymbol><ci id="S4.SS2.p2.3.m3.1.1.2.cmml" xref="S4.SS2.p2.3.m3.1.1.2">ğ•</ci><ci id="S4.SS2.p2.3.m3.1.1.3.cmml" xref="S4.SS2.p2.3.m3.1.1.3">ğ‘—</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p2.3.m3.1c">\mathbf{V}_{j}</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p2.3.m3.1d">bold_V start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT</annotation></semantics></math> based on attention scores derived from the scoring function <math alttext="s(\mathbf{Q}_{i},\mathbf{K}_{j})" class="ltx_Math" display="inline" id="S4.SS2.p2.4.m4.2"><semantics id="S4.SS2.p2.4.m4.2a"><mrow id="S4.SS2.p2.4.m4.2.2" xref="S4.SS2.p2.4.m4.2.2.cmml"><mi id="S4.SS2.p2.4.m4.2.2.4" xref="S4.SS2.p2.4.m4.2.2.4.cmml">s</mi><mo id="S4.SS2.p2.4.m4.2.2.3" xref="S4.SS2.p2.4.m4.2.2.3.cmml">â¢</mo><mrow id="S4.SS2.p2.4.m4.2.2.2.2" xref="S4.SS2.p2.4.m4.2.2.2.3.cmml"><mo id="S4.SS2.p2.4.m4.2.2.2.2.3" stretchy="false" xref="S4.SS2.p2.4.m4.2.2.2.3.cmml">(</mo><msub id="S4.SS2.p2.4.m4.1.1.1.1.1" xref="S4.SS2.p2.4.m4.1.1.1.1.1.cmml"><mi id="S4.SS2.p2.4.m4.1.1.1.1.1.2" xref="S4.SS2.p2.4.m4.1.1.1.1.1.2.cmml">ğ</mi><mi id="S4.SS2.p2.4.m4.1.1.1.1.1.3" xref="S4.SS2.p2.4.m4.1.1.1.1.1.3.cmml">i</mi></msub><mo id="S4.SS2.p2.4.m4.2.2.2.2.4" xref="S4.SS2.p2.4.m4.2.2.2.3.cmml">,</mo><msub id="S4.SS2.p2.4.m4.2.2.2.2.2" xref="S4.SS2.p2.4.m4.2.2.2.2.2.cmml"><mi id="S4.SS2.p2.4.m4.2.2.2.2.2.2" xref="S4.SS2.p2.4.m4.2.2.2.2.2.2.cmml">ğŠ</mi><mi id="S4.SS2.p2.4.m4.2.2.2.2.2.3" xref="S4.SS2.p2.4.m4.2.2.2.2.2.3.cmml">j</mi></msub><mo id="S4.SS2.p2.4.m4.2.2.2.2.5" stretchy="false" xref="S4.SS2.p2.4.m4.2.2.2.3.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S4.SS2.p2.4.m4.2b"><apply id="S4.SS2.p2.4.m4.2.2.cmml" xref="S4.SS2.p2.4.m4.2.2"><times id="S4.SS2.p2.4.m4.2.2.3.cmml" xref="S4.SS2.p2.4.m4.2.2.3"></times><ci id="S4.SS2.p2.4.m4.2.2.4.cmml" xref="S4.SS2.p2.4.m4.2.2.4">ğ‘ </ci><interval closure="open" id="S4.SS2.p2.4.m4.2.2.2.3.cmml" xref="S4.SS2.p2.4.m4.2.2.2.2"><apply id="S4.SS2.p2.4.m4.1.1.1.1.1.cmml" xref="S4.SS2.p2.4.m4.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.SS2.p2.4.m4.1.1.1.1.1.1.cmml" xref="S4.SS2.p2.4.m4.1.1.1.1.1">subscript</csymbol><ci id="S4.SS2.p2.4.m4.1.1.1.1.1.2.cmml" xref="S4.SS2.p2.4.m4.1.1.1.1.1.2">ğ</ci><ci id="S4.SS2.p2.4.m4.1.1.1.1.1.3.cmml" xref="S4.SS2.p2.4.m4.1.1.1.1.1.3">ğ‘–</ci></apply><apply id="S4.SS2.p2.4.m4.2.2.2.2.2.cmml" xref="S4.SS2.p2.4.m4.2.2.2.2.2"><csymbol cd="ambiguous" id="S4.SS2.p2.4.m4.2.2.2.2.2.1.cmml" xref="S4.SS2.p2.4.m4.2.2.2.2.2">subscript</csymbol><ci id="S4.SS2.p2.4.m4.2.2.2.2.2.2.cmml" xref="S4.SS2.p2.4.m4.2.2.2.2.2.2">ğŠ</ci><ci id="S4.SS2.p2.4.m4.2.2.2.2.2.3.cmml" xref="S4.SS2.p2.4.m4.2.2.2.2.2.3">ğ‘—</ci></apply></interval></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p2.4.m4.2c">s(\mathbf{Q}_{i},\mathbf{K}_{j})</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p2.4.m4.2d">italic_s ( bold_Q start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT , bold_K start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT )</annotation></semantics></math>:</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS2.p3">
<table class="ltx_equation ltx_eqn_table" id="S4.E1">
<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math alttext="\mathbf{y}_{i}=\sum_{j=1}^{n}\text{softmax}\left(s(\mathbf{Q}_{i},\mathbf{K}_{%
j})\right)\mathbf{V}_{j}." class="ltx_Math" display="block" id="S4.E1.m1.1"><semantics id="S4.E1.m1.1a"><mrow id="S4.E1.m1.1.1.1" xref="S4.E1.m1.1.1.1.1.cmml"><mrow id="S4.E1.m1.1.1.1.1" xref="S4.E1.m1.1.1.1.1.cmml"><msub id="S4.E1.m1.1.1.1.1.3" xref="S4.E1.m1.1.1.1.1.3.cmml"><mi id="S4.E1.m1.1.1.1.1.3.2" xref="S4.E1.m1.1.1.1.1.3.2.cmml">ğ²</mi><mi id="S4.E1.m1.1.1.1.1.3.3" xref="S4.E1.m1.1.1.1.1.3.3.cmml">i</mi></msub><mo id="S4.E1.m1.1.1.1.1.2" rspace="0.111em" xref="S4.E1.m1.1.1.1.1.2.cmml">=</mo><mrow id="S4.E1.m1.1.1.1.1.1" xref="S4.E1.m1.1.1.1.1.1.cmml"><munderover id="S4.E1.m1.1.1.1.1.1.2" xref="S4.E1.m1.1.1.1.1.1.2.cmml"><mo id="S4.E1.m1.1.1.1.1.1.2.2.2" movablelimits="false" xref="S4.E1.m1.1.1.1.1.1.2.2.2.cmml">âˆ‘</mo><mrow id="S4.E1.m1.1.1.1.1.1.2.2.3" xref="S4.E1.m1.1.1.1.1.1.2.2.3.cmml"><mi id="S4.E1.m1.1.1.1.1.1.2.2.3.2" xref="S4.E1.m1.1.1.1.1.1.2.2.3.2.cmml">j</mi><mo id="S4.E1.m1.1.1.1.1.1.2.2.3.1" xref="S4.E1.m1.1.1.1.1.1.2.2.3.1.cmml">=</mo><mn id="S4.E1.m1.1.1.1.1.1.2.2.3.3" xref="S4.E1.m1.1.1.1.1.1.2.2.3.3.cmml">1</mn></mrow><mi id="S4.E1.m1.1.1.1.1.1.2.3" xref="S4.E1.m1.1.1.1.1.1.2.3.cmml">n</mi></munderover><mrow id="S4.E1.m1.1.1.1.1.1.1" xref="S4.E1.m1.1.1.1.1.1.1.cmml"><mtext id="S4.E1.m1.1.1.1.1.1.1.3" xref="S4.E1.m1.1.1.1.1.1.1.3a.cmml">softmax</mtext><mo id="S4.E1.m1.1.1.1.1.1.1.2" xref="S4.E1.m1.1.1.1.1.1.1.2.cmml">â¢</mo><mrow id="S4.E1.m1.1.1.1.1.1.1.1.1" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.cmml"><mo id="S4.E1.m1.1.1.1.1.1.1.1.1.2" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.cmml">(</mo><mrow id="S4.E1.m1.1.1.1.1.1.1.1.1.1" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.E1.m1.1.1.1.1.1.1.1.1.1.4" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.4.cmml">s</mi><mo id="S4.E1.m1.1.1.1.1.1.1.1.1.1.3" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.3.cmml">â¢</mo><mrow id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml"><mo id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.3" stretchy="false" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">(</mo><msub id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2.cmml">ğ</mi><mi id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3.cmml">i</mi></msub><mo id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.4" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">,</mo><msub id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.cmml"><mi id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2.cmml">ğŠ</mi><mi id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3.cmml">j</mi></msub><mo id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.5" stretchy="false" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">)</mo></mrow></mrow><mo id="S4.E1.m1.1.1.1.1.1.1.1.1.3" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.cmml">)</mo></mrow><mo id="S4.E1.m1.1.1.1.1.1.1.2a" xref="S4.E1.m1.1.1.1.1.1.1.2.cmml">â¢</mo><msub id="S4.E1.m1.1.1.1.1.1.1.4" xref="S4.E1.m1.1.1.1.1.1.1.4.cmml"><mi id="S4.E1.m1.1.1.1.1.1.1.4.2" xref="S4.E1.m1.1.1.1.1.1.1.4.2.cmml">ğ•</mi><mi id="S4.E1.m1.1.1.1.1.1.1.4.3" xref="S4.E1.m1.1.1.1.1.1.1.4.3.cmml">j</mi></msub></mrow></mrow></mrow><mo id="S4.E1.m1.1.1.1.2" lspace="0em" xref="S4.E1.m1.1.1.1.1.cmml">.</mo></mrow><annotation-xml encoding="MathML-Content" id="S4.E1.m1.1b"><apply id="S4.E1.m1.1.1.1.1.cmml" xref="S4.E1.m1.1.1.1"><eq id="S4.E1.m1.1.1.1.1.2.cmml" xref="S4.E1.m1.1.1.1.1.2"></eq><apply id="S4.E1.m1.1.1.1.1.3.cmml" xref="S4.E1.m1.1.1.1.1.3"><csymbol cd="ambiguous" id="S4.E1.m1.1.1.1.1.3.1.cmml" xref="S4.E1.m1.1.1.1.1.3">subscript</csymbol><ci id="S4.E1.m1.1.1.1.1.3.2.cmml" xref="S4.E1.m1.1.1.1.1.3.2">ğ²</ci><ci id="S4.E1.m1.1.1.1.1.3.3.cmml" xref="S4.E1.m1.1.1.1.1.3.3">ğ‘–</ci></apply><apply id="S4.E1.m1.1.1.1.1.1.cmml" xref="S4.E1.m1.1.1.1.1.1"><apply id="S4.E1.m1.1.1.1.1.1.2.cmml" xref="S4.E1.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S4.E1.m1.1.1.1.1.1.2.1.cmml" xref="S4.E1.m1.1.1.1.1.1.2">superscript</csymbol><apply id="S4.E1.m1.1.1.1.1.1.2.2.cmml" xref="S4.E1.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S4.E1.m1.1.1.1.1.1.2.2.1.cmml" xref="S4.E1.m1.1.1.1.1.1.2">subscript</csymbol><sum id="S4.E1.m1.1.1.1.1.1.2.2.2.cmml" xref="S4.E1.m1.1.1.1.1.1.2.2.2"></sum><apply id="S4.E1.m1.1.1.1.1.1.2.2.3.cmml" xref="S4.E1.m1.1.1.1.1.1.2.2.3"><eq id="S4.E1.m1.1.1.1.1.1.2.2.3.1.cmml" xref="S4.E1.m1.1.1.1.1.1.2.2.3.1"></eq><ci id="S4.E1.m1.1.1.1.1.1.2.2.3.2.cmml" xref="S4.E1.m1.1.1.1.1.1.2.2.3.2">ğ‘—</ci><cn id="S4.E1.m1.1.1.1.1.1.2.2.3.3.cmml" type="integer" xref="S4.E1.m1.1.1.1.1.1.2.2.3.3">1</cn></apply></apply><ci id="S4.E1.m1.1.1.1.1.1.2.3.cmml" xref="S4.E1.m1.1.1.1.1.1.2.3">ğ‘›</ci></apply><apply id="S4.E1.m1.1.1.1.1.1.1.cmml" xref="S4.E1.m1.1.1.1.1.1.1"><times id="S4.E1.m1.1.1.1.1.1.1.2.cmml" xref="S4.E1.m1.1.1.1.1.1.1.2"></times><ci id="S4.E1.m1.1.1.1.1.1.1.3a.cmml" xref="S4.E1.m1.1.1.1.1.1.1.3"><mtext id="S4.E1.m1.1.1.1.1.1.1.3.cmml" xref="S4.E1.m1.1.1.1.1.1.1.3">softmax</mtext></ci><apply id="S4.E1.m1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1"><times id="S4.E1.m1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.3"></times><ci id="S4.E1.m1.1.1.1.1.1.1.1.1.1.4.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.4">ğ‘ </ci><interval closure="open" id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2"><apply id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2">ğ</ci><ci id="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3">ğ‘–</ci></apply><apply id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2"><csymbol cd="ambiguous" id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.1.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2">subscript</csymbol><ci id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2">ğŠ</ci><ci id="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3.cmml" xref="S4.E1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3">ğ‘—</ci></apply></interval></apply><apply id="S4.E1.m1.1.1.1.1.1.1.4.cmml" xref="S4.E1.m1.1.1.1.1.1.1.4"><csymbol cd="ambiguous" id="S4.E1.m1.1.1.1.1.1.1.4.1.cmml" xref="S4.E1.m1.1.1.1.1.1.1.4">subscript</csymbol><ci id="S4.E1.m1.1.1.1.1.1.1.4.2.cmml" xref="S4.E1.m1.1.1.1.1.1.1.4.2">ğ•</ci><ci id="S4.E1.m1.1.1.1.1.1.1.4.3.cmml" xref="S4.E1.m1.1.1.1.1.1.1.4.3">ğ‘—</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.E1.m1.1c">\mathbf{y}_{i}=\sum_{j=1}^{n}\text{softmax}\left(s(\mathbf{Q}_{i},\mathbf{K}_{%
j})\right)\mathbf{V}_{j}.</annotation><annotation encoding="application/x-llamapun" id="S4.E1.m1.1d">bold_y start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT = âˆ‘ start_POSTSUBSCRIPT italic_j = 1 end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_n end_POSTSUPERSCRIPT softmax ( italic_s ( bold_Q start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT , bold_K start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT ) ) bold_V start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT .</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right" rowspan="1"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
<p class="ltx_p" id="S4.SS2.p3.2">In this formulation, the softmax function normalises scores, facilitating the modelâ€™s emphasis on specific patches. The resultant <math alttext="\mathbf{y}_{i}" class="ltx_Math" display="inline" id="S4.SS2.p3.1.m1.1"><semantics id="S4.SS2.p3.1.m1.1a"><msub id="S4.SS2.p3.1.m1.1.1" xref="S4.SS2.p3.1.m1.1.1.cmml"><mi id="S4.SS2.p3.1.m1.1.1.2" xref="S4.SS2.p3.1.m1.1.1.2.cmml">ğ²</mi><mi id="S4.SS2.p3.1.m1.1.1.3" xref="S4.SS2.p3.1.m1.1.1.3.cmml">i</mi></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p3.1.m1.1b"><apply id="S4.SS2.p3.1.m1.1.1.cmml" xref="S4.SS2.p3.1.m1.1.1"><csymbol cd="ambiguous" id="S4.SS2.p3.1.m1.1.1.1.cmml" xref="S4.SS2.p3.1.m1.1.1">subscript</csymbol><ci id="S4.SS2.p3.1.m1.1.1.2.cmml" xref="S4.SS2.p3.1.m1.1.1.2">ğ²</ci><ci id="S4.SS2.p3.1.m1.1.1.3.cmml" xref="S4.SS2.p3.1.m1.1.1.3">ğ‘–</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p3.1.m1.1c">\mathbf{y}_{i}</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p3.1.m1.1d">bold_y start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT</annotation></semantics></math> is a weighted blend of patch-level Value vectors <math alttext="\mathbf{V}_{j}" class="ltx_Math" display="inline" id="S4.SS2.p3.2.m2.1"><semantics id="S4.SS2.p3.2.m2.1a"><msub id="S4.SS2.p3.2.m2.1.1" xref="S4.SS2.p3.2.m2.1.1.cmml"><mi id="S4.SS2.p3.2.m2.1.1.2" xref="S4.SS2.p3.2.m2.1.1.2.cmml">ğ•</mi><mi id="S4.SS2.p3.2.m2.1.1.3" xref="S4.SS2.p3.2.m2.1.1.3.cmml">j</mi></msub><annotation-xml encoding="MathML-Content" id="S4.SS2.p3.2.m2.1b"><apply id="S4.SS2.p3.2.m2.1.1.cmml" xref="S4.SS2.p3.2.m2.1.1"><csymbol cd="ambiguous" id="S4.SS2.p3.2.m2.1.1.1.cmml" xref="S4.SS2.p3.2.m2.1.1">subscript</csymbol><ci id="S4.SS2.p3.2.m2.1.1.2.cmml" xref="S4.SS2.p3.2.m2.1.1.2">ğ•</ci><ci id="S4.SS2.p3.2.m2.1.1.3.cmml" xref="S4.SS2.p3.2.m2.1.1.3">ğ‘—</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS2.p3.2.m2.1c">\mathbf{V}_{j}</annotation><annotation encoding="application/x-llamapun" id="S4.SS2.p3.2.m2.1d">bold_V start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT</annotation></semantics></math>, representing the patient-level feature.</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS2.p4">
<p class="ltx_p" id="S4.SS2.p4.1">Two primary advantages emerge:</p>
<ol class="ltx_enumerate" id="S4.I1">
<li class="ltx_item" id="S4.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para" id="S4.I1.i1.p1">
<p class="ltx_p" id="S4.I1.i1.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I1.i1.p1.1.1">Inherent Feature Importance:</span> Using image patches for Key, Query and Value enables the model to naturally highlight significant regions in histopathological images, capturing inherent sample heterogeneity.</p>
</div>
</li>
<li class="ltx_item" id="S4.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para ltx_noindent" id="S4.I1.i2.p1">
<p class="ltx_p" id="S4.I1.i2.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I1.i2.p1.1.1">Inter-Patch Contextualisation:</span> The mechanism grasps the context among distinct patches, contextualising each patch with respect to others for a comprehensive patient-level representation.</p>
</div>
</li>
</ol>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS2.p5">
<p class="ltx_p" id="S4.SS2.p5.1">The enriched patient-level vector, emphasising both context and focus, is pivotal for subsequent survival risk prediction tasks.</p>
</div>
<figure class="ltx_table" id="S4.T1">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_table">Table 1: </span>Performance Comparison using C-index on TCGA Data.</figcaption>
<table class="ltx_tabular ltx_centering ltx_guessed_headers ltx_align_middle" id="S4.T1.7">
<tbody class="ltx_tbody">
<tr class="ltx_tr" id="S4.T1.7.8.1">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_tt" id="S4.T1.7.8.1.1">CV Fold</th>
<td class="ltx_td ltx_align_center ltx_border_tt" colspan="2" id="S4.T1.7.8.1.2">Multimodal (Imaging+Genetic+Clinical)</td>
<td class="ltx_td ltx_align_center ltx_border_tt" colspan="2" id="S4.T1.7.8.1.3">Imaging+Genetic</td>
<td class="ltx_td ltx_align_center ltx_border_tt" colspan="2" id="S4.T1.7.8.1.4">Imaging</td>
<td class="ltx_td ltx_align_center ltx_border_tt" id="S4.T1.7.8.1.5">Clinical</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.9.2">
<th class="ltx_td ltx_th ltx_th_row" id="S4.T1.7.9.2.1"></th>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.2">MaxViT</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.3">ResNet50</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.4">MaxViT</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.5">ResNet50</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.6">MaxViT</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.7">ResNet50</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.9.2.8">CoxPH</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.10.3">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_t" id="S4.T1.7.10.3.1">1</th>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.2">0.70</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.3">0.64</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.4">0.72</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.5">0.61</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.6">0.45</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.7">0.47</td>
<td class="ltx_td ltx_align_center ltx_border_t" id="S4.T1.7.10.3.8">0.50</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.11.4">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S4.T1.7.11.4.1">2</th>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.2">0.57</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.3">0.49</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.4">0.44</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.5">0.58</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.6">0.47</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.7">0.47</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.11.4.8">0.24</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.12.5">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S4.T1.7.12.5.1">3</th>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.2">0.66</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.3">0.62</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.4">0.63</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.5">0.65</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.6">0.55</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.7">0.57</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.12.5.8">0.34</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.13.6">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S4.T1.7.13.6.1">4</th>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.2">0.62</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.3">0.55</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.4">0.50</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.5">0.48</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.6">0.60</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.7">0.60</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.13.6.8">0.57</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.14.7">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row" id="S4.T1.7.14.7.1">5</th>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.2">0.70</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.3">0.50</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.4">0.57</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.5">0.51</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.6">0.62</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.7">0.65</td>
<td class="ltx_td ltx_align_center" id="S4.T1.7.14.7.8">0.71</td>
</tr>
<tr class="ltx_tr" id="S4.T1.7.7">
<th class="ltx_td ltx_align_center ltx_th ltx_th_row ltx_border_bb ltx_border_t" id="S4.T1.7.7.8">Mean</th>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.1.1.1"><math alttext="0.64\pm 0.06" class="ltx_Math" display="inline" id="S4.T1.1.1.1.m1.1"><semantics id="S4.T1.1.1.1.m1.1a"><mrow id="S4.T1.1.1.1.m1.1.1" xref="S4.T1.1.1.1.m1.1.1.cmml"><mn id="S4.T1.1.1.1.m1.1.1.2" xref="S4.T1.1.1.1.m1.1.1.2.cmml">0.64</mn><mo id="S4.T1.1.1.1.m1.1.1.1" xref="S4.T1.1.1.1.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.1.1.1.m1.1.1.3" xref="S4.T1.1.1.1.m1.1.1.3.cmml">0.06</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.1.1.1.m1.1b"><apply id="S4.T1.1.1.1.m1.1.1.cmml" xref="S4.T1.1.1.1.m1.1.1"><csymbol cd="latexml" id="S4.T1.1.1.1.m1.1.1.1.cmml" xref="S4.T1.1.1.1.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.1.1.1.m1.1.1.2.cmml" type="float" xref="S4.T1.1.1.1.m1.1.1.2">0.64</cn><cn id="S4.T1.1.1.1.m1.1.1.3.cmml" type="float" xref="S4.T1.1.1.1.m1.1.1.3">0.06</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.1.1.1.m1.1c">0.64\pm 0.06</annotation><annotation encoding="application/x-llamapun" id="S4.T1.1.1.1.m1.1d">0.64 Â± 0.06</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.2.2.2"><math alttext="0.56\pm 0.06" class="ltx_Math" display="inline" id="S4.T1.2.2.2.m1.1"><semantics id="S4.T1.2.2.2.m1.1a"><mrow id="S4.T1.2.2.2.m1.1.1" xref="S4.T1.2.2.2.m1.1.1.cmml"><mn id="S4.T1.2.2.2.m1.1.1.2" xref="S4.T1.2.2.2.m1.1.1.2.cmml">0.56</mn><mo id="S4.T1.2.2.2.m1.1.1.1" xref="S4.T1.2.2.2.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.2.2.2.m1.1.1.3" xref="S4.T1.2.2.2.m1.1.1.3.cmml">0.06</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.2.2.2.m1.1b"><apply id="S4.T1.2.2.2.m1.1.1.cmml" xref="S4.T1.2.2.2.m1.1.1"><csymbol cd="latexml" id="S4.T1.2.2.2.m1.1.1.1.cmml" xref="S4.T1.2.2.2.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.2.2.2.m1.1.1.2.cmml" type="float" xref="S4.T1.2.2.2.m1.1.1.2">0.56</cn><cn id="S4.T1.2.2.2.m1.1.1.3.cmml" type="float" xref="S4.T1.2.2.2.m1.1.1.3">0.06</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.2.2.2.m1.1c">0.56\pm 0.06</annotation><annotation encoding="application/x-llamapun" id="S4.T1.2.2.2.m1.1d">0.56 Â± 0.06</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.3.3.3"><math alttext="0.57\pm 0.10" class="ltx_Math" display="inline" id="S4.T1.3.3.3.m1.1"><semantics id="S4.T1.3.3.3.m1.1a"><mrow id="S4.T1.3.3.3.m1.1.1" xref="S4.T1.3.3.3.m1.1.1.cmml"><mn id="S4.T1.3.3.3.m1.1.1.2" xref="S4.T1.3.3.3.m1.1.1.2.cmml">0.57</mn><mo id="S4.T1.3.3.3.m1.1.1.1" xref="S4.T1.3.3.3.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.3.3.3.m1.1.1.3" xref="S4.T1.3.3.3.m1.1.1.3.cmml">0.10</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.3.3.3.m1.1b"><apply id="S4.T1.3.3.3.m1.1.1.cmml" xref="S4.T1.3.3.3.m1.1.1"><csymbol cd="latexml" id="S4.T1.3.3.3.m1.1.1.1.cmml" xref="S4.T1.3.3.3.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.3.3.3.m1.1.1.2.cmml" type="float" xref="S4.T1.3.3.3.m1.1.1.2">0.57</cn><cn id="S4.T1.3.3.3.m1.1.1.3.cmml" type="float" xref="S4.T1.3.3.3.m1.1.1.3">0.10</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.3.3.3.m1.1c">0.57\pm 0.10</annotation><annotation encoding="application/x-llamapun" id="S4.T1.3.3.3.m1.1d">0.57 Â± 0.10</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.4.4.4"><math alttext="0.56\pm 0.06" class="ltx_Math" display="inline" id="S4.T1.4.4.4.m1.1"><semantics id="S4.T1.4.4.4.m1.1a"><mrow id="S4.T1.4.4.4.m1.1.1" xref="S4.T1.4.4.4.m1.1.1.cmml"><mn id="S4.T1.4.4.4.m1.1.1.2" xref="S4.T1.4.4.4.m1.1.1.2.cmml">0.56</mn><mo id="S4.T1.4.4.4.m1.1.1.1" xref="S4.T1.4.4.4.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.4.4.4.m1.1.1.3" xref="S4.T1.4.4.4.m1.1.1.3.cmml">0.06</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.4.4.4.m1.1b"><apply id="S4.T1.4.4.4.m1.1.1.cmml" xref="S4.T1.4.4.4.m1.1.1"><csymbol cd="latexml" id="S4.T1.4.4.4.m1.1.1.1.cmml" xref="S4.T1.4.4.4.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.4.4.4.m1.1.1.2.cmml" type="float" xref="S4.T1.4.4.4.m1.1.1.2">0.56</cn><cn id="S4.T1.4.4.4.m1.1.1.3.cmml" type="float" xref="S4.T1.4.4.4.m1.1.1.3">0.06</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.4.4.4.m1.1c">0.56\pm 0.06</annotation><annotation encoding="application/x-llamapun" id="S4.T1.4.4.4.m1.1d">0.56 Â± 0.06</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.5.5.5"><math alttext="0.53\pm 0.08" class="ltx_Math" display="inline" id="S4.T1.5.5.5.m1.1"><semantics id="S4.T1.5.5.5.m1.1a"><mrow id="S4.T1.5.5.5.m1.1.1" xref="S4.T1.5.5.5.m1.1.1.cmml"><mn id="S4.T1.5.5.5.m1.1.1.2" xref="S4.T1.5.5.5.m1.1.1.2.cmml">0.53</mn><mo id="S4.T1.5.5.5.m1.1.1.1" xref="S4.T1.5.5.5.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.5.5.5.m1.1.1.3" xref="S4.T1.5.5.5.m1.1.1.3.cmml">0.08</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.5.5.5.m1.1b"><apply id="S4.T1.5.5.5.m1.1.1.cmml" xref="S4.T1.5.5.5.m1.1.1"><csymbol cd="latexml" id="S4.T1.5.5.5.m1.1.1.1.cmml" xref="S4.T1.5.5.5.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.5.5.5.m1.1.1.2.cmml" type="float" xref="S4.T1.5.5.5.m1.1.1.2">0.53</cn><cn id="S4.T1.5.5.5.m1.1.1.3.cmml" type="float" xref="S4.T1.5.5.5.m1.1.1.3">0.08</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.5.5.5.m1.1c">0.53\pm 0.08</annotation><annotation encoding="application/x-llamapun" id="S4.T1.5.5.5.m1.1d">0.53 Â± 0.08</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.6.6.6"><math alttext="0.55\pm 0.08" class="ltx_Math" display="inline" id="S4.T1.6.6.6.m1.1"><semantics id="S4.T1.6.6.6.m1.1a"><mrow id="S4.T1.6.6.6.m1.1.1" xref="S4.T1.6.6.6.m1.1.1.cmml"><mn id="S4.T1.6.6.6.m1.1.1.2" xref="S4.T1.6.6.6.m1.1.1.2.cmml">0.55</mn><mo id="S4.T1.6.6.6.m1.1.1.1" xref="S4.T1.6.6.6.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.6.6.6.m1.1.1.3" xref="S4.T1.6.6.6.m1.1.1.3.cmml">0.08</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.6.6.6.m1.1b"><apply id="S4.T1.6.6.6.m1.1.1.cmml" xref="S4.T1.6.6.6.m1.1.1"><csymbol cd="latexml" id="S4.T1.6.6.6.m1.1.1.1.cmml" xref="S4.T1.6.6.6.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.6.6.6.m1.1.1.2.cmml" type="float" xref="S4.T1.6.6.6.m1.1.1.2">0.55</cn><cn id="S4.T1.6.6.6.m1.1.1.3.cmml" type="float" xref="S4.T1.6.6.6.m1.1.1.3">0.08</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.6.6.6.m1.1c">0.55\pm 0.08</annotation><annotation encoding="application/x-llamapun" id="S4.T1.6.6.6.m1.1d">0.55 Â± 0.08</annotation></semantics></math></td>
<td class="ltx_td ltx_align_center ltx_border_bb ltx_border_t" id="S4.T1.7.7.7">
<math alttext="0.47\pm 0.17" class="ltx_Math" display="inline" id="S4.T1.7.7.7.m1.1"><semantics id="S4.T1.7.7.7.m1.1a"><mrow id="S4.T1.7.7.7.m1.1.1" xref="S4.T1.7.7.7.m1.1.1.cmml"><mn id="S4.T1.7.7.7.m1.1.1.2" xref="S4.T1.7.7.7.m1.1.1.2.cmml">0.47</mn><mo id="S4.T1.7.7.7.m1.1.1.1" xref="S4.T1.7.7.7.m1.1.1.1.cmml">Â±</mo><mn id="S4.T1.7.7.7.m1.1.1.3" xref="S4.T1.7.7.7.m1.1.1.3.cmml">0.17</mn></mrow><annotation-xml encoding="MathML-Content" id="S4.T1.7.7.7.m1.1b"><apply id="S4.T1.7.7.7.m1.1.1.cmml" xref="S4.T1.7.7.7.m1.1.1"><csymbol cd="latexml" id="S4.T1.7.7.7.m1.1.1.1.cmml" xref="S4.T1.7.7.7.m1.1.1.1">plus-or-minus</csymbol><cn id="S4.T1.7.7.7.m1.1.1.2.cmml" type="float" xref="S4.T1.7.7.7.m1.1.1.2">0.47</cn><cn id="S4.T1.7.7.7.m1.1.1.3.cmml" type="float" xref="S4.T1.7.7.7.m1.1.1.3">0.17</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.T1.7.7.7.m1.1c">0.47\pm 0.17</annotation><annotation encoding="application/x-llamapun" id="S4.T1.7.7.7.m1.1d">0.47 Â± 0.17</annotation></semantics></math></td>
</tr>
</tbody>
</table>
</figure>
<figure class="ltx_figure" id="S4.F4"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="424" id="S4.F4.g1" src="x4.png" width="830"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>The proposed MM-SurvNet architecture employs a multimodal deep learning framework for survival risk prediction in cancer patients. Image embedding is concatenated with genetic data using dual cross-attention mechanisms and further integrated with clinical data in the final layer.</figcaption>
</figure>
</section>
<section class="ltx_subsection" id="S4.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.3 </span>Multimodal Fusion Using Dual Cross-Attention</h3>
<div class="ltx_para ltx_noindent" id="S4.SS3.p1">
<p class="ltx_p" id="S4.SS3.p1.1">Fusing data from diverse sources is paramount in multimodal learning. In our architecture, we deploy a dual cross-attention mechanism to achieve a refined interaction between histopathology image features and genetic expression profiles. Specifically, this encompasses two attention operations:</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS3.p2">
<ol class="ltx_enumerate" id="S4.I2">
<li class="ltx_item" id="S4.I2.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para" id="S4.I2.i1.p1">
<p class="ltx_p" id="S4.I2.i1.p1.1">Image features serve as the Query (Q) while genetic features act as both the Key (K) and Value (V).</p>
</div>
</li>
<li class="ltx_item" id="S4.I2.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para ltx_noindent" id="S4.I2.i2.p1">
<p class="ltx_p" id="S4.I2.i2.p1.1">Conversely, genetic features form the Query, with image features becoming the Key and Value.</p>
</div>
</li>
</ol>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS3.p3">
<p class="ltx_p" id="S4.SS3.p3.3">Let <math alttext="\mathbf{I}=\{\mathbf{i}_{1},\mathbf{i}_{2},\ldots,\mathbf{i}_{n}\}" class="ltx_Math" display="inline" id="S4.SS3.p3.1.m1.4"><semantics id="S4.SS3.p3.1.m1.4a"><mrow id="S4.SS3.p3.1.m1.4.4" xref="S4.SS3.p3.1.m1.4.4.cmml"><mi id="S4.SS3.p3.1.m1.4.4.5" xref="S4.SS3.p3.1.m1.4.4.5.cmml">ğˆ</mi><mo id="S4.SS3.p3.1.m1.4.4.4" xref="S4.SS3.p3.1.m1.4.4.4.cmml">=</mo><mrow id="S4.SS3.p3.1.m1.4.4.3.3" xref="S4.SS3.p3.1.m1.4.4.3.4.cmml"><mo id="S4.SS3.p3.1.m1.4.4.3.3.4" stretchy="false" xref="S4.SS3.p3.1.m1.4.4.3.4.cmml">{</mo><msub id="S4.SS3.p3.1.m1.2.2.1.1.1" xref="S4.SS3.p3.1.m1.2.2.1.1.1.cmml"><mi id="S4.SS3.p3.1.m1.2.2.1.1.1.2" xref="S4.SS3.p3.1.m1.2.2.1.1.1.2.cmml">ğ¢</mi><mn id="S4.SS3.p3.1.m1.2.2.1.1.1.3" xref="S4.SS3.p3.1.m1.2.2.1.1.1.3.cmml">1</mn></msub><mo id="S4.SS3.p3.1.m1.4.4.3.3.5" xref="S4.SS3.p3.1.m1.4.4.3.4.cmml">,</mo><msub id="S4.SS3.p3.1.m1.3.3.2.2.2" xref="S4.SS3.p3.1.m1.3.3.2.2.2.cmml"><mi id="S4.SS3.p3.1.m1.3.3.2.2.2.2" xref="S4.SS3.p3.1.m1.3.3.2.2.2.2.cmml">ğ¢</mi><mn id="S4.SS3.p3.1.m1.3.3.2.2.2.3" xref="S4.SS3.p3.1.m1.3.3.2.2.2.3.cmml">2</mn></msub><mo id="S4.SS3.p3.1.m1.4.4.3.3.6" xref="S4.SS3.p3.1.m1.4.4.3.4.cmml">,</mo><mi id="S4.SS3.p3.1.m1.1.1" mathvariant="normal" xref="S4.SS3.p3.1.m1.1.1.cmml">â€¦</mi><mo id="S4.SS3.p3.1.m1.4.4.3.3.7" xref="S4.SS3.p3.1.m1.4.4.3.4.cmml">,</mo><msub id="S4.SS3.p3.1.m1.4.4.3.3.3" xref="S4.SS3.p3.1.m1.4.4.3.3.3.cmml"><mi id="S4.SS3.p3.1.m1.4.4.3.3.3.2" xref="S4.SS3.p3.1.m1.4.4.3.3.3.2.cmml">ğ¢</mi><mi id="S4.SS3.p3.1.m1.4.4.3.3.3.3" xref="S4.SS3.p3.1.m1.4.4.3.3.3.3.cmml">n</mi></msub><mo id="S4.SS3.p3.1.m1.4.4.3.3.8" stretchy="false" xref="S4.SS3.p3.1.m1.4.4.3.4.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S4.SS3.p3.1.m1.4b"><apply id="S4.SS3.p3.1.m1.4.4.cmml" xref="S4.SS3.p3.1.m1.4.4"><eq id="S4.SS3.p3.1.m1.4.4.4.cmml" xref="S4.SS3.p3.1.m1.4.4.4"></eq><ci id="S4.SS3.p3.1.m1.4.4.5.cmml" xref="S4.SS3.p3.1.m1.4.4.5">ğˆ</ci><set id="S4.SS3.p3.1.m1.4.4.3.4.cmml" xref="S4.SS3.p3.1.m1.4.4.3.3"><apply id="S4.SS3.p3.1.m1.2.2.1.1.1.cmml" xref="S4.SS3.p3.1.m1.2.2.1.1.1"><csymbol cd="ambiguous" id="S4.SS3.p3.1.m1.2.2.1.1.1.1.cmml" xref="S4.SS3.p3.1.m1.2.2.1.1.1">subscript</csymbol><ci id="S4.SS3.p3.1.m1.2.2.1.1.1.2.cmml" xref="S4.SS3.p3.1.m1.2.2.1.1.1.2">ğ¢</ci><cn id="S4.SS3.p3.1.m1.2.2.1.1.1.3.cmml" type="integer" xref="S4.SS3.p3.1.m1.2.2.1.1.1.3">1</cn></apply><apply id="S4.SS3.p3.1.m1.3.3.2.2.2.cmml" xref="S4.SS3.p3.1.m1.3.3.2.2.2"><csymbol cd="ambiguous" id="S4.SS3.p3.1.m1.3.3.2.2.2.1.cmml" xref="S4.SS3.p3.1.m1.3.3.2.2.2">subscript</csymbol><ci id="S4.SS3.p3.1.m1.3.3.2.2.2.2.cmml" xref="S4.SS3.p3.1.m1.3.3.2.2.2.2">ğ¢</ci><cn id="S4.SS3.p3.1.m1.3.3.2.2.2.3.cmml" type="integer" xref="S4.SS3.p3.1.m1.3.3.2.2.2.3">2</cn></apply><ci id="S4.SS3.p3.1.m1.1.1.cmml" xref="S4.SS3.p3.1.m1.1.1">â€¦</ci><apply id="S4.SS3.p3.1.m1.4.4.3.3.3.cmml" xref="S4.SS3.p3.1.m1.4.4.3.3.3"><csymbol cd="ambiguous" id="S4.SS3.p3.1.m1.4.4.3.3.3.1.cmml" xref="S4.SS3.p3.1.m1.4.4.3.3.3">subscript</csymbol><ci id="S4.SS3.p3.1.m1.4.4.3.3.3.2.cmml" xref="S4.SS3.p3.1.m1.4.4.3.3.3.2">ğ¢</ci><ci id="S4.SS3.p3.1.m1.4.4.3.3.3.3.cmml" xref="S4.SS3.p3.1.m1.4.4.3.3.3.3">ğ‘›</ci></apply></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p3.1.m1.4c">\mathbf{I}=\{\mathbf{i}_{1},\mathbf{i}_{2},\ldots,\mathbf{i}_{n}\}</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p3.1.m1.4d">bold_I = { bold_i start_POSTSUBSCRIPT 1 end_POSTSUBSCRIPT , bold_i start_POSTSUBSCRIPT 2 end_POSTSUBSCRIPT , â€¦ , bold_i start_POSTSUBSCRIPT italic_n end_POSTSUBSCRIPT }</annotation></semantics></math> be the set of image feature vectors and <math alttext="\mathbf{G}=\{\mathbf{g}_{1},\mathbf{g}_{2},\ldots,\mathbf{g}_{m}\}" class="ltx_Math" display="inline" id="S4.SS3.p3.2.m2.4"><semantics id="S4.SS3.p3.2.m2.4a"><mrow id="S4.SS3.p3.2.m2.4.4" xref="S4.SS3.p3.2.m2.4.4.cmml"><mi id="S4.SS3.p3.2.m2.4.4.5" xref="S4.SS3.p3.2.m2.4.4.5.cmml">ğ†</mi><mo id="S4.SS3.p3.2.m2.4.4.4" xref="S4.SS3.p3.2.m2.4.4.4.cmml">=</mo><mrow id="S4.SS3.p3.2.m2.4.4.3.3" xref="S4.SS3.p3.2.m2.4.4.3.4.cmml"><mo id="S4.SS3.p3.2.m2.4.4.3.3.4" stretchy="false" xref="S4.SS3.p3.2.m2.4.4.3.4.cmml">{</mo><msub id="S4.SS3.p3.2.m2.2.2.1.1.1" xref="S4.SS3.p3.2.m2.2.2.1.1.1.cmml"><mi id="S4.SS3.p3.2.m2.2.2.1.1.1.2" xref="S4.SS3.p3.2.m2.2.2.1.1.1.2.cmml">ğ </mi><mn id="S4.SS3.p3.2.m2.2.2.1.1.1.3" xref="S4.SS3.p3.2.m2.2.2.1.1.1.3.cmml">1</mn></msub><mo id="S4.SS3.p3.2.m2.4.4.3.3.5" xref="S4.SS3.p3.2.m2.4.4.3.4.cmml">,</mo><msub id="S4.SS3.p3.2.m2.3.3.2.2.2" xref="S4.SS3.p3.2.m2.3.3.2.2.2.cmml"><mi id="S4.SS3.p3.2.m2.3.3.2.2.2.2" xref="S4.SS3.p3.2.m2.3.3.2.2.2.2.cmml">ğ </mi><mn id="S4.SS3.p3.2.m2.3.3.2.2.2.3" xref="S4.SS3.p3.2.m2.3.3.2.2.2.3.cmml">2</mn></msub><mo id="S4.SS3.p3.2.m2.4.4.3.3.6" xref="S4.SS3.p3.2.m2.4.4.3.4.cmml">,</mo><mi id="S4.SS3.p3.2.m2.1.1" mathvariant="normal" xref="S4.SS3.p3.2.m2.1.1.cmml">â€¦</mi><mo id="S4.SS3.p3.2.m2.4.4.3.3.7" xref="S4.SS3.p3.2.m2.4.4.3.4.cmml">,</mo><msub id="S4.SS3.p3.2.m2.4.4.3.3.3" xref="S4.SS3.p3.2.m2.4.4.3.3.3.cmml"><mi id="S4.SS3.p3.2.m2.4.4.3.3.3.2" xref="S4.SS3.p3.2.m2.4.4.3.3.3.2.cmml">ğ </mi><mi id="S4.SS3.p3.2.m2.4.4.3.3.3.3" xref="S4.SS3.p3.2.m2.4.4.3.3.3.3.cmml">m</mi></msub><mo id="S4.SS3.p3.2.m2.4.4.3.3.8" stretchy="false" xref="S4.SS3.p3.2.m2.4.4.3.4.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S4.SS3.p3.2.m2.4b"><apply id="S4.SS3.p3.2.m2.4.4.cmml" xref="S4.SS3.p3.2.m2.4.4"><eq id="S4.SS3.p3.2.m2.4.4.4.cmml" xref="S4.SS3.p3.2.m2.4.4.4"></eq><ci id="S4.SS3.p3.2.m2.4.4.5.cmml" xref="S4.SS3.p3.2.m2.4.4.5">ğ†</ci><set id="S4.SS3.p3.2.m2.4.4.3.4.cmml" xref="S4.SS3.p3.2.m2.4.4.3.3"><apply id="S4.SS3.p3.2.m2.2.2.1.1.1.cmml" xref="S4.SS3.p3.2.m2.2.2.1.1.1"><csymbol cd="ambiguous" id="S4.SS3.p3.2.m2.2.2.1.1.1.1.cmml" xref="S4.SS3.p3.2.m2.2.2.1.1.1">subscript</csymbol><ci id="S4.SS3.p3.2.m2.2.2.1.1.1.2.cmml" xref="S4.SS3.p3.2.m2.2.2.1.1.1.2">ğ </ci><cn id="S4.SS3.p3.2.m2.2.2.1.1.1.3.cmml" type="integer" xref="S4.SS3.p3.2.m2.2.2.1.1.1.3">1</cn></apply><apply id="S4.SS3.p3.2.m2.3.3.2.2.2.cmml" xref="S4.SS3.p3.2.m2.3.3.2.2.2"><csymbol cd="ambiguous" id="S4.SS3.p3.2.m2.3.3.2.2.2.1.cmml" xref="S4.SS3.p3.2.m2.3.3.2.2.2">subscript</csymbol><ci id="S4.SS3.p3.2.m2.3.3.2.2.2.2.cmml" xref="S4.SS3.p3.2.m2.3.3.2.2.2.2">ğ </ci><cn id="S4.SS3.p3.2.m2.3.3.2.2.2.3.cmml" type="integer" xref="S4.SS3.p3.2.m2.3.3.2.2.2.3">2</cn></apply><ci id="S4.SS3.p3.2.m2.1.1.cmml" xref="S4.SS3.p3.2.m2.1.1">â€¦</ci><apply id="S4.SS3.p3.2.m2.4.4.3.3.3.cmml" xref="S4.SS3.p3.2.m2.4.4.3.3.3"><csymbol cd="ambiguous" id="S4.SS3.p3.2.m2.4.4.3.3.3.1.cmml" xref="S4.SS3.p3.2.m2.4.4.3.3.3">subscript</csymbol><ci id="S4.SS3.p3.2.m2.4.4.3.3.3.2.cmml" xref="S4.SS3.p3.2.m2.4.4.3.3.3.2">ğ </ci><ci id="S4.SS3.p3.2.m2.4.4.3.3.3.3.cmml" xref="S4.SS3.p3.2.m2.4.4.3.3.3.3">ğ‘š</ci></apply></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p3.2.m2.4c">\mathbf{G}=\{\mathbf{g}_{1},\mathbf{g}_{2},\ldots,\mathbf{g}_{m}\}</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p3.2.m2.4d">bold_G = { bold_g start_POSTSUBSCRIPT 1 end_POSTSUBSCRIPT , bold_g start_POSTSUBSCRIPT 2 end_POSTSUBSCRIPT , â€¦ , bold_g start_POSTSUBSCRIPT italic_m end_POSTSUBSCRIPT }</annotation></semantics></math> denote the genetic feature vectors. The enriched feature vectors for the <math alttext="k^{\text{th}}" class="ltx_Math" display="inline" id="S4.SS3.p3.3.m3.1"><semantics id="S4.SS3.p3.3.m3.1a"><msup id="S4.SS3.p3.3.m3.1.1" xref="S4.SS3.p3.3.m3.1.1.cmml"><mi id="S4.SS3.p3.3.m3.1.1.2" xref="S4.SS3.p3.3.m3.1.1.2.cmml">k</mi><mtext id="S4.SS3.p3.3.m3.1.1.3" xref="S4.SS3.p3.3.m3.1.1.3a.cmml">th</mtext></msup><annotation-xml encoding="MathML-Content" id="S4.SS3.p3.3.m3.1b"><apply id="S4.SS3.p3.3.m3.1.1.cmml" xref="S4.SS3.p3.3.m3.1.1"><csymbol cd="ambiguous" id="S4.SS3.p3.3.m3.1.1.1.cmml" xref="S4.SS3.p3.3.m3.1.1">superscript</csymbol><ci id="S4.SS3.p3.3.m3.1.1.2.cmml" xref="S4.SS3.p3.3.m3.1.1.2">ğ‘˜</ci><ci id="S4.SS3.p3.3.m3.1.1.3a.cmml" xref="S4.SS3.p3.3.m3.1.1.3"><mtext id="S4.SS3.p3.3.m3.1.1.3.cmml" mathsize="70%" xref="S4.SS3.p3.3.m3.1.1.3">th</mtext></ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p3.3.m3.1c">k^{\text{th}}</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p3.3.m3.1d">italic_k start_POSTSUPERSCRIPT th end_POSTSUPERSCRIPT</annotation></semantics></math> image and genetic patches are:</p>
<table class="ltx_equation ltx_eqn_table" id="S4.Ex1">
<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math alttext="\mathbf{y}^{I}_{k}=\sum_{j=1}^{m}\text{softmax}\left(s(\mathbf{i}_{k},\mathbf{%
g}_{j})\right)\mathbf{g}_{j}," class="ltx_Math" display="block" id="S4.Ex1.m1.1"><semantics id="S4.Ex1.m1.1a"><mrow id="S4.Ex1.m1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.cmml"><mrow id="S4.Ex1.m1.1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.cmml"><msubsup id="S4.Ex1.m1.1.1.1.1.3" xref="S4.Ex1.m1.1.1.1.1.3.cmml"><mi id="S4.Ex1.m1.1.1.1.1.3.2.2" xref="S4.Ex1.m1.1.1.1.1.3.2.2.cmml">ğ²</mi><mi id="S4.Ex1.m1.1.1.1.1.3.3" xref="S4.Ex1.m1.1.1.1.1.3.3.cmml">k</mi><mi id="S4.Ex1.m1.1.1.1.1.3.2.3" xref="S4.Ex1.m1.1.1.1.1.3.2.3.cmml">I</mi></msubsup><mo id="S4.Ex1.m1.1.1.1.1.2" rspace="0.111em" xref="S4.Ex1.m1.1.1.1.1.2.cmml">=</mo><mrow id="S4.Ex1.m1.1.1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.1.cmml"><munderover id="S4.Ex1.m1.1.1.1.1.1.2" xref="S4.Ex1.m1.1.1.1.1.1.2.cmml"><mo id="S4.Ex1.m1.1.1.1.1.1.2.2.2" movablelimits="false" xref="S4.Ex1.m1.1.1.1.1.1.2.2.2.cmml">âˆ‘</mo><mrow id="S4.Ex1.m1.1.1.1.1.1.2.2.3" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.cmml"><mi id="S4.Ex1.m1.1.1.1.1.1.2.2.3.2" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.2.cmml">j</mi><mo id="S4.Ex1.m1.1.1.1.1.1.2.2.3.1" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.1.cmml">=</mo><mn id="S4.Ex1.m1.1.1.1.1.1.2.2.3.3" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.3.cmml">1</mn></mrow><mi id="S4.Ex1.m1.1.1.1.1.1.2.3" xref="S4.Ex1.m1.1.1.1.1.1.2.3.cmml">m</mi></munderover><mrow id="S4.Ex1.m1.1.1.1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.1.1.cmml"><mtext id="S4.Ex1.m1.1.1.1.1.1.1.3" xref="S4.Ex1.m1.1.1.1.1.1.1.3a.cmml">softmax</mtext><mo id="S4.Ex1.m1.1.1.1.1.1.1.2" xref="S4.Ex1.m1.1.1.1.1.1.1.2.cmml">â¢</mo><mrow id="S4.Ex1.m1.1.1.1.1.1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.cmml"><mo id="S4.Ex1.m1.1.1.1.1.1.1.1.1.2" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.cmml">(</mo><mrow id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.4" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.4.cmml">s</mi><mo id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.3" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.3.cmml">â¢</mo><mrow id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml"><mo id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.3" stretchy="false" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">(</mo><msub id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2.cmml">ğ¢</mi><mi id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3.cmml">k</mi></msub><mo id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.4" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">,</mo><msub id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.cmml"><mi id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2.cmml">ğ </mi><mi id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3.cmml">j</mi></msub><mo id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.5" stretchy="false" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">)</mo></mrow></mrow><mo id="S4.Ex1.m1.1.1.1.1.1.1.1.1.3" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.cmml">)</mo></mrow><mo id="S4.Ex1.m1.1.1.1.1.1.1.2a" xref="S4.Ex1.m1.1.1.1.1.1.1.2.cmml">â¢</mo><msub id="S4.Ex1.m1.1.1.1.1.1.1.4" xref="S4.Ex1.m1.1.1.1.1.1.1.4.cmml"><mi id="S4.Ex1.m1.1.1.1.1.1.1.4.2" xref="S4.Ex1.m1.1.1.1.1.1.1.4.2.cmml">ğ </mi><mi id="S4.Ex1.m1.1.1.1.1.1.1.4.3" xref="S4.Ex1.m1.1.1.1.1.1.1.4.3.cmml">j</mi></msub></mrow></mrow></mrow><mo id="S4.Ex1.m1.1.1.1.2" xref="S4.Ex1.m1.1.1.1.1.cmml">,</mo></mrow><annotation-xml encoding="MathML-Content" id="S4.Ex1.m1.1b"><apply id="S4.Ex1.m1.1.1.1.1.cmml" xref="S4.Ex1.m1.1.1.1"><eq id="S4.Ex1.m1.1.1.1.1.2.cmml" xref="S4.Ex1.m1.1.1.1.1.2"></eq><apply id="S4.Ex1.m1.1.1.1.1.3.cmml" xref="S4.Ex1.m1.1.1.1.1.3"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.3.1.cmml" xref="S4.Ex1.m1.1.1.1.1.3">subscript</csymbol><apply id="S4.Ex1.m1.1.1.1.1.3.2.cmml" xref="S4.Ex1.m1.1.1.1.1.3"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.3.2.1.cmml" xref="S4.Ex1.m1.1.1.1.1.3">superscript</csymbol><ci id="S4.Ex1.m1.1.1.1.1.3.2.2.cmml" xref="S4.Ex1.m1.1.1.1.1.3.2.2">ğ²</ci><ci id="S4.Ex1.m1.1.1.1.1.3.2.3.cmml" xref="S4.Ex1.m1.1.1.1.1.3.2.3">ğ¼</ci></apply><ci id="S4.Ex1.m1.1.1.1.1.3.3.cmml" xref="S4.Ex1.m1.1.1.1.1.3.3">ğ‘˜</ci></apply><apply id="S4.Ex1.m1.1.1.1.1.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1"><apply id="S4.Ex1.m1.1.1.1.1.1.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.1.2.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2">superscript</csymbol><apply id="S4.Ex1.m1.1.1.1.1.1.2.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.1.2.2.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2">subscript</csymbol><sum id="S4.Ex1.m1.1.1.1.1.1.2.2.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2.2.2"></sum><apply id="S4.Ex1.m1.1.1.1.1.1.2.2.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3"><eq id="S4.Ex1.m1.1.1.1.1.1.2.2.3.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.1"></eq><ci id="S4.Ex1.m1.1.1.1.1.1.2.2.3.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.2">ğ‘—</ci><cn id="S4.Ex1.m1.1.1.1.1.1.2.2.3.3.cmml" type="integer" xref="S4.Ex1.m1.1.1.1.1.1.2.2.3.3">1</cn></apply></apply><ci id="S4.Ex1.m1.1.1.1.1.1.2.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.2.3">ğ‘š</ci></apply><apply id="S4.Ex1.m1.1.1.1.1.1.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1"><times id="S4.Ex1.m1.1.1.1.1.1.1.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.2"></times><ci id="S4.Ex1.m1.1.1.1.1.1.1.3a.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.3"><mtext id="S4.Ex1.m1.1.1.1.1.1.1.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.3">softmax</mtext></ci><apply id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1"><times id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.3"></times><ci id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.4.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.4">ğ‘ </ci><interval closure="open" id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2"><apply id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.2">ğ¢</ci><ci id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.1.1.1.3">ğ‘˜</ci></apply><apply id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2">subscript</csymbol><ci id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.2">ğ </ci><ci id="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.1.1.1.2.2.2.3">ğ‘—</ci></apply></interval></apply><apply id="S4.Ex1.m1.1.1.1.1.1.1.4.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.4"><csymbol cd="ambiguous" id="S4.Ex1.m1.1.1.1.1.1.1.4.1.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.4">subscript</csymbol><ci id="S4.Ex1.m1.1.1.1.1.1.1.4.2.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.4.2">ğ </ci><ci id="S4.Ex1.m1.1.1.1.1.1.1.4.3.cmml" xref="S4.Ex1.m1.1.1.1.1.1.1.4.3">ğ‘—</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.Ex1.m1.1c">\mathbf{y}^{I}_{k}=\sum_{j=1}^{m}\text{softmax}\left(s(\mathbf{i}_{k},\mathbf{%
g}_{j})\right)\mathbf{g}_{j},</annotation><annotation encoding="application/x-llamapun" id="S4.Ex1.m1.1d">bold_y start_POSTSUPERSCRIPT italic_I end_POSTSUPERSCRIPT start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT = âˆ‘ start_POSTSUBSCRIPT italic_j = 1 end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_m end_POSTSUPERSCRIPT softmax ( italic_s ( bold_i start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT , bold_g start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT ) ) bold_g start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT ,</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
<table class="ltx_equation ltx_eqn_table" id="S4.Ex2">
<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math alttext="\mathbf{y}^{G}_{k}=\sum_{i=1}^{n}\text{softmax}\left(s(\mathbf{g}_{k},\mathbf{%
i}_{i})\right)\mathbf{i}_{i}." class="ltx_Math" display="block" id="S4.Ex2.m1.1"><semantics id="S4.Ex2.m1.1a"><mrow id="S4.Ex2.m1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.cmml"><mrow id="S4.Ex2.m1.1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.cmml"><msubsup id="S4.Ex2.m1.1.1.1.1.3" xref="S4.Ex2.m1.1.1.1.1.3.cmml"><mi id="S4.Ex2.m1.1.1.1.1.3.2.2" xref="S4.Ex2.m1.1.1.1.1.3.2.2.cmml">ğ²</mi><mi id="S4.Ex2.m1.1.1.1.1.3.3" xref="S4.Ex2.m1.1.1.1.1.3.3.cmml">k</mi><mi id="S4.Ex2.m1.1.1.1.1.3.2.3" xref="S4.Ex2.m1.1.1.1.1.3.2.3.cmml">G</mi></msubsup><mo id="S4.Ex2.m1.1.1.1.1.2" rspace="0.111em" xref="S4.Ex2.m1.1.1.1.1.2.cmml">=</mo><mrow id="S4.Ex2.m1.1.1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.1.cmml"><munderover id="S4.Ex2.m1.1.1.1.1.1.2" xref="S4.Ex2.m1.1.1.1.1.1.2.cmml"><mo id="S4.Ex2.m1.1.1.1.1.1.2.2.2" movablelimits="false" xref="S4.Ex2.m1.1.1.1.1.1.2.2.2.cmml">âˆ‘</mo><mrow id="S4.Ex2.m1.1.1.1.1.1.2.2.3" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.cmml"><mi id="S4.Ex2.m1.1.1.1.1.1.2.2.3.2" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.2.cmml">i</mi><mo id="S4.Ex2.m1.1.1.1.1.1.2.2.3.1" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.1.cmml">=</mo><mn id="S4.Ex2.m1.1.1.1.1.1.2.2.3.3" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.3.cmml">1</mn></mrow><mi id="S4.Ex2.m1.1.1.1.1.1.2.3" xref="S4.Ex2.m1.1.1.1.1.1.2.3.cmml">n</mi></munderover><mrow id="S4.Ex2.m1.1.1.1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.1.1.cmml"><mtext id="S4.Ex2.m1.1.1.1.1.1.1.3" xref="S4.Ex2.m1.1.1.1.1.1.1.3a.cmml">softmax</mtext><mo id="S4.Ex2.m1.1.1.1.1.1.1.2" xref="S4.Ex2.m1.1.1.1.1.1.1.2.cmml">â¢</mo><mrow id="S4.Ex2.m1.1.1.1.1.1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.cmml"><mo id="S4.Ex2.m1.1.1.1.1.1.1.1.1.2" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.cmml">(</mo><mrow id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.4" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.4.cmml">s</mi><mo id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.3" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.3.cmml">â¢</mo><mrow id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.3.cmml"><mo id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.3" stretchy="false" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">(</mo><msub id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.2" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.2.cmml">ğ </mi><mi id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.3" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.3.cmml">k</mi></msub><mo id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.4" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">,</mo><msub id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.cmml"><mi id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.2" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.2.cmml">ğ¢</mi><mi id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.3" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.3.cmml">i</mi></msub><mo id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.5" stretchy="false" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.3.cmml">)</mo></mrow></mrow><mo id="S4.Ex2.m1.1.1.1.1.1.1.1.1.3" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.cmml">)</mo></mrow><mo id="S4.Ex2.m1.1.1.1.1.1.1.2a" xref="S4.Ex2.m1.1.1.1.1.1.1.2.cmml">â¢</mo><msub id="S4.Ex2.m1.1.1.1.1.1.1.4" xref="S4.Ex2.m1.1.1.1.1.1.1.4.cmml"><mi id="S4.Ex2.m1.1.1.1.1.1.1.4.2" xref="S4.Ex2.m1.1.1.1.1.1.1.4.2.cmml">ğ¢</mi><mi id="S4.Ex2.m1.1.1.1.1.1.1.4.3" xref="S4.Ex2.m1.1.1.1.1.1.1.4.3.cmml">i</mi></msub></mrow></mrow></mrow><mo id="S4.Ex2.m1.1.1.1.2" lspace="0em" xref="S4.Ex2.m1.1.1.1.1.cmml">.</mo></mrow><annotation-xml encoding="MathML-Content" id="S4.Ex2.m1.1b"><apply id="S4.Ex2.m1.1.1.1.1.cmml" xref="S4.Ex2.m1.1.1.1"><eq id="S4.Ex2.m1.1.1.1.1.2.cmml" xref="S4.Ex2.m1.1.1.1.1.2"></eq><apply id="S4.Ex2.m1.1.1.1.1.3.cmml" xref="S4.Ex2.m1.1.1.1.1.3"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.3.1.cmml" xref="S4.Ex2.m1.1.1.1.1.3">subscript</csymbol><apply id="S4.Ex2.m1.1.1.1.1.3.2.cmml" xref="S4.Ex2.m1.1.1.1.1.3"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.3.2.1.cmml" xref="S4.Ex2.m1.1.1.1.1.3">superscript</csymbol><ci id="S4.Ex2.m1.1.1.1.1.3.2.2.cmml" xref="S4.Ex2.m1.1.1.1.1.3.2.2">ğ²</ci><ci id="S4.Ex2.m1.1.1.1.1.3.2.3.cmml" xref="S4.Ex2.m1.1.1.1.1.3.2.3">ğº</ci></apply><ci id="S4.Ex2.m1.1.1.1.1.3.3.cmml" xref="S4.Ex2.m1.1.1.1.1.3.3">ğ‘˜</ci></apply><apply id="S4.Ex2.m1.1.1.1.1.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1"><apply id="S4.Ex2.m1.1.1.1.1.1.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.1.2.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2">superscript</csymbol><apply id="S4.Ex2.m1.1.1.1.1.1.2.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.1.2.2.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2">subscript</csymbol><sum id="S4.Ex2.m1.1.1.1.1.1.2.2.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2.2.2"></sum><apply id="S4.Ex2.m1.1.1.1.1.1.2.2.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3"><eq id="S4.Ex2.m1.1.1.1.1.1.2.2.3.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.1"></eq><ci id="S4.Ex2.m1.1.1.1.1.1.2.2.3.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.2">ğ‘–</ci><cn id="S4.Ex2.m1.1.1.1.1.1.2.2.3.3.cmml" type="integer" xref="S4.Ex2.m1.1.1.1.1.1.2.2.3.3">1</cn></apply></apply><ci id="S4.Ex2.m1.1.1.1.1.1.2.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.2.3">ğ‘›</ci></apply><apply id="S4.Ex2.m1.1.1.1.1.1.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1"><times id="S4.Ex2.m1.1.1.1.1.1.1.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.2"></times><ci id="S4.Ex2.m1.1.1.1.1.1.1.3a.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.3"><mtext id="S4.Ex2.m1.1.1.1.1.1.1.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.3">softmax</mtext></ci><apply id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1"><times id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.3"></times><ci id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.4.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.4">ğ‘ </ci><interval closure="open" id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2"><apply id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.2">ğ </ci><ci id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.1.1.1.3">ğ‘˜</ci></apply><apply id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2">subscript</csymbol><ci id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.2">ğ¢</ci><ci id="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.1.1.1.2.2.2.3">ğ‘–</ci></apply></interval></apply><apply id="S4.Ex2.m1.1.1.1.1.1.1.4.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.4"><csymbol cd="ambiguous" id="S4.Ex2.m1.1.1.1.1.1.1.4.1.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.4">subscript</csymbol><ci id="S4.Ex2.m1.1.1.1.1.1.1.4.2.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.4.2">ğ¢</ci><ci id="S4.Ex2.m1.1.1.1.1.1.1.4.3.cmml" xref="S4.Ex2.m1.1.1.1.1.1.1.4.3">ğ‘–</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.Ex2.m1.1c">\mathbf{y}^{G}_{k}=\sum_{i=1}^{n}\text{softmax}\left(s(\mathbf{g}_{k},\mathbf{%
i}_{i})\right)\mathbf{i}_{i}.</annotation><annotation encoding="application/x-llamapun" id="S4.Ex2.m1.1d">bold_y start_POSTSUPERSCRIPT italic_G end_POSTSUPERSCRIPT start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT = âˆ‘ start_POSTSUBSCRIPT italic_i = 1 end_POSTSUBSCRIPT start_POSTSUPERSCRIPT italic_n end_POSTSUPERSCRIPT softmax ( italic_s ( bold_g start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT , bold_i start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT ) ) bold_i start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT .</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
<p class="ltx_p" id="S4.SS3.p3.4">Here, the scoring function <math alttext="s(.)" class="ltx_math_unparsed" display="inline" id="S4.SS3.p3.4.m1.1"><semantics id="S4.SS3.p3.4.m1.1a"><mrow id="S4.SS3.p3.4.m1.1b"><mi id="S4.SS3.p3.4.m1.1.1">s</mi><mrow id="S4.SS3.p3.4.m1.1.2"><mo id="S4.SS3.p3.4.m1.1.2.1" stretchy="false">(</mo><mo id="S4.SS3.p3.4.m1.1.2.2" lspace="0em" rspace="0.167em">.</mo><mo id="S4.SS3.p3.4.m1.1.2.3" stretchy="false">)</mo></mrow></mrow><annotation encoding="application/x-tex" id="S4.SS3.p3.4.m1.1c">s(.)</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p3.4.m1.1d">italic_s ( . )</annotation></semantics></math> gauges the relevance between Queries and Keys. These weighted sums, after being normalised by the softmax function, are then concatenated to form a unified feature representation:</p>
<table class="ltx_equation ltx_eqn_table" id="S4.Ex3">
<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math alttext="\mathbf{y}_{k}=\text{Concatenate}(\mathbf{y}^{I}_{k},\mathbf{y}^{G}_{k})" class="ltx_Math" display="block" id="S4.Ex3.m1.2"><semantics id="S4.Ex3.m1.2a"><mrow id="S4.Ex3.m1.2.2" xref="S4.Ex3.m1.2.2.cmml"><msub id="S4.Ex3.m1.2.2.4" xref="S4.Ex3.m1.2.2.4.cmml"><mi id="S4.Ex3.m1.2.2.4.2" xref="S4.Ex3.m1.2.2.4.2.cmml">ğ²</mi><mi id="S4.Ex3.m1.2.2.4.3" xref="S4.Ex3.m1.2.2.4.3.cmml">k</mi></msub><mo id="S4.Ex3.m1.2.2.3" xref="S4.Ex3.m1.2.2.3.cmml">=</mo><mrow id="S4.Ex3.m1.2.2.2" xref="S4.Ex3.m1.2.2.2.cmml"><mtext id="S4.Ex3.m1.2.2.2.4" xref="S4.Ex3.m1.2.2.2.4a.cmml">Concatenate</mtext><mo id="S4.Ex3.m1.2.2.2.3" xref="S4.Ex3.m1.2.2.2.3.cmml">â¢</mo><mrow id="S4.Ex3.m1.2.2.2.2.2" xref="S4.Ex3.m1.2.2.2.2.3.cmml"><mo id="S4.Ex3.m1.2.2.2.2.2.3" stretchy="false" xref="S4.Ex3.m1.2.2.2.2.3.cmml">(</mo><msubsup id="S4.Ex3.m1.1.1.1.1.1.1" xref="S4.Ex3.m1.1.1.1.1.1.1.cmml"><mi id="S4.Ex3.m1.1.1.1.1.1.1.2.2" xref="S4.Ex3.m1.1.1.1.1.1.1.2.2.cmml">ğ²</mi><mi id="S4.Ex3.m1.1.1.1.1.1.1.3" xref="S4.Ex3.m1.1.1.1.1.1.1.3.cmml">k</mi><mi id="S4.Ex3.m1.1.1.1.1.1.1.2.3" xref="S4.Ex3.m1.1.1.1.1.1.1.2.3.cmml">I</mi></msubsup><mo id="S4.Ex3.m1.2.2.2.2.2.4" xref="S4.Ex3.m1.2.2.2.2.3.cmml">,</mo><msubsup id="S4.Ex3.m1.2.2.2.2.2.2" xref="S4.Ex3.m1.2.2.2.2.2.2.cmml"><mi id="S4.Ex3.m1.2.2.2.2.2.2.2.2" xref="S4.Ex3.m1.2.2.2.2.2.2.2.2.cmml">ğ²</mi><mi id="S4.Ex3.m1.2.2.2.2.2.2.3" xref="S4.Ex3.m1.2.2.2.2.2.2.3.cmml">k</mi><mi id="S4.Ex3.m1.2.2.2.2.2.2.2.3" xref="S4.Ex3.m1.2.2.2.2.2.2.2.3.cmml">G</mi></msubsup><mo id="S4.Ex3.m1.2.2.2.2.2.5" stretchy="false" xref="S4.Ex3.m1.2.2.2.2.3.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S4.Ex3.m1.2b"><apply id="S4.Ex3.m1.2.2.cmml" xref="S4.Ex3.m1.2.2"><eq id="S4.Ex3.m1.2.2.3.cmml" xref="S4.Ex3.m1.2.2.3"></eq><apply id="S4.Ex3.m1.2.2.4.cmml" xref="S4.Ex3.m1.2.2.4"><csymbol cd="ambiguous" id="S4.Ex3.m1.2.2.4.1.cmml" xref="S4.Ex3.m1.2.2.4">subscript</csymbol><ci id="S4.Ex3.m1.2.2.4.2.cmml" xref="S4.Ex3.m1.2.2.4.2">ğ²</ci><ci id="S4.Ex3.m1.2.2.4.3.cmml" xref="S4.Ex3.m1.2.2.4.3">ğ‘˜</ci></apply><apply id="S4.Ex3.m1.2.2.2.cmml" xref="S4.Ex3.m1.2.2.2"><times id="S4.Ex3.m1.2.2.2.3.cmml" xref="S4.Ex3.m1.2.2.2.3"></times><ci id="S4.Ex3.m1.2.2.2.4a.cmml" xref="S4.Ex3.m1.2.2.2.4"><mtext id="S4.Ex3.m1.2.2.2.4.cmml" xref="S4.Ex3.m1.2.2.2.4">Concatenate</mtext></ci><interval closure="open" id="S4.Ex3.m1.2.2.2.2.3.cmml" xref="S4.Ex3.m1.2.2.2.2.2"><apply id="S4.Ex3.m1.1.1.1.1.1.1.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.Ex3.m1.1.1.1.1.1.1.1.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1">subscript</csymbol><apply id="S4.Ex3.m1.1.1.1.1.1.1.2.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.Ex3.m1.1.1.1.1.1.1.2.1.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1">superscript</csymbol><ci id="S4.Ex3.m1.1.1.1.1.1.1.2.2.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1.2.2">ğ²</ci><ci id="S4.Ex3.m1.1.1.1.1.1.1.2.3.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1.2.3">ğ¼</ci></apply><ci id="S4.Ex3.m1.1.1.1.1.1.1.3.cmml" xref="S4.Ex3.m1.1.1.1.1.1.1.3">ğ‘˜</ci></apply><apply id="S4.Ex3.m1.2.2.2.2.2.2.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2"><csymbol cd="ambiguous" id="S4.Ex3.m1.2.2.2.2.2.2.1.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2">subscript</csymbol><apply id="S4.Ex3.m1.2.2.2.2.2.2.2.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2"><csymbol cd="ambiguous" id="S4.Ex3.m1.2.2.2.2.2.2.2.1.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2">superscript</csymbol><ci id="S4.Ex3.m1.2.2.2.2.2.2.2.2.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2.2.2">ğ²</ci><ci id="S4.Ex3.m1.2.2.2.2.2.2.2.3.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2.2.3">ğº</ci></apply><ci id="S4.Ex3.m1.2.2.2.2.2.2.3.cmml" xref="S4.Ex3.m1.2.2.2.2.2.2.3">ğ‘˜</ci></apply></interval></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.Ex3.m1.2c">\mathbf{y}_{k}=\text{Concatenate}(\mathbf{y}^{I}_{k},\mathbf{y}^{G}_{k})</annotation><annotation encoding="application/x-llamapun" id="S4.Ex3.m1.2d">bold_y start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT = Concatenate ( bold_y start_POSTSUPERSCRIPT italic_I end_POSTSUPERSCRIPT start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT , bold_y start_POSTSUPERSCRIPT italic_G end_POSTSUPERSCRIPT start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT )</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
</tr></tbody>
</table>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS3.p4">
<p class="ltx_p" id="S4.SS3.p4.1">The dual cross-attention offers two primary advantages:</p>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS3.p5">
<ol class="ltx_enumerate" id="S4.I3">
<li class="ltx_item" id="S4.I3.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">1.</span>
<div class="ltx_para ltx_noindent" id="S4.I3.i1.p1">
<p class="ltx_p" id="S4.I3.i1.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I3.i1.p1.1.1">Bidirectional Contextualisation:</span> The mechanism captures intermodal dependencies in both directionsâ€”image to genetic and vice versaâ€”providing a holistic contextualised representation.</p>
</div>
</li>
<li class="ltx_item" id="S4.I3.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">2.</span>
<div class="ltx_para ltx_noindent" id="S4.I3.i2.p1">
<p class="ltx_p" id="S4.I3.i2.p1.1"><span class="ltx_text ltx_font_bold" id="S4.I3.i2.p1.1.1">Enhanced Selective Attention:</span> By alternating the role of Query, the architecture gleans specific features from both modalities that are crucial for each other, ensuring compact and informative fused feature vectors.</p>
</div>
</li>
</ol>
</div>
<div class="ltx_para ltx_noindent" id="S4.SS3.p6">
<p class="ltx_p" id="S4.SS3.p6.1">The resultant vectors <math alttext="\mathbf{y}_{k}" class="ltx_Math" display="inline" id="S4.SS3.p6.1.m1.1"><semantics id="S4.SS3.p6.1.m1.1a"><msub id="S4.SS3.p6.1.m1.1.1" xref="S4.SS3.p6.1.m1.1.1.cmml"><mi id="S4.SS3.p6.1.m1.1.1.2" xref="S4.SS3.p6.1.m1.1.1.2.cmml">ğ²</mi><mi id="S4.SS3.p6.1.m1.1.1.3" xref="S4.SS3.p6.1.m1.1.1.3.cmml">k</mi></msub><annotation-xml encoding="MathML-Content" id="S4.SS3.p6.1.m1.1b"><apply id="S4.SS3.p6.1.m1.1.1.cmml" xref="S4.SS3.p6.1.m1.1.1"><csymbol cd="ambiguous" id="S4.SS3.p6.1.m1.1.1.1.cmml" xref="S4.SS3.p6.1.m1.1.1">subscript</csymbol><ci id="S4.SS3.p6.1.m1.1.1.2.cmml" xref="S4.SS3.p6.1.m1.1.1.2">ğ²</ci><ci id="S4.SS3.p6.1.m1.1.1.3.cmml" xref="S4.SS3.p6.1.m1.1.1.3">ğ‘˜</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS3.p6.1.m1.1c">\mathbf{y}_{k}</annotation><annotation encoding="application/x-llamapun" id="S4.SS3.p6.1.m1.1d">bold_y start_POSTSUBSCRIPT italic_k end_POSTSUBSCRIPT</annotation></semantics></math> are thus contextually enriched and optimised for tasks such as survival risk stratification.</p>
</div>
</section>
<section class="ltx_subsection" id="S4.SS4">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.4 </span>MM-SurvNet Model Training</h3>
<div class="ltx_para ltx_noindent" id="S4.SS4.p1">
<p class="ltx_p" id="S4.SS4.p1.4">The MM-SurvNet model (Fig.Â <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S3.F3" title="Figure 3 â€£ 3 Data Preparation â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">3</span></a>) employs either a pretrained Vision Transformer (MaxViT) or a CNN (ResNet50) for histopathological image feature extraction. The multimodal data is subsequently incorporated into the MM-SurvNet model (Fig.Â <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.F4" title="Figure 4 â€£ 4.2 Patch-to-Patient Aggregation Using Self-Attention â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">4</span></a>) to enhance prediction accuracy. The Adam optimiser with decoupled weight decay (AdamW) with a learning rate of 0.001 and a mini-batch size of 12 governs the optimisation, which includes an early stopping criterion set at a patience of 5 and a maximum of 150 epochsÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib25" title="">25</a>]</cite>. The primary loss function is the negative log-likelihood (NLL), aligning with our survival analysis objectives, and optimisation targets the concordance index (C-index) as the key performance metric<cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib26" title="">26</a>]</cite>. The NLL loss is calculated as:</p>
<table class="ltx_equation ltx_eqn_table" id="S4.E2">
<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math alttext="\text{Loss}=-\frac{\sum_{i}\left(\log(h_{i})-\log\left(\sum_{j\in R_{i}}\exp(%
\log(h_{j}))\right)\right)\cdot d_{i}}{\sum_{i}d_{i}}" class="ltx_Math" display="block" id="S4.E2.m1.5"><semantics id="S4.E2.m1.5a"><mrow id="S4.E2.m1.5.6" xref="S4.E2.m1.5.6.cmml"><mtext id="S4.E2.m1.5.6.2" xref="S4.E2.m1.5.6.2a.cmml">Loss</mtext><mo id="S4.E2.m1.5.6.1" xref="S4.E2.m1.5.6.1.cmml">=</mo><mrow id="S4.E2.m1.5.6.3" xref="S4.E2.m1.5.6.3.cmml"><mo id="S4.E2.m1.5.6.3a" xref="S4.E2.m1.5.6.3.cmml">âˆ’</mo><mfrac id="S4.E2.m1.5.5" xref="S4.E2.m1.5.5.cmml"><mrow id="S4.E2.m1.5.5.5" xref="S4.E2.m1.5.5.5.cmml"><msub id="S4.E2.m1.5.5.5.6" xref="S4.E2.m1.5.5.5.6.cmml"><mo id="S4.E2.m1.5.5.5.6.2" xref="S4.E2.m1.5.5.5.6.2.cmml">âˆ‘</mo><mi id="S4.E2.m1.5.5.5.6.3" xref="S4.E2.m1.5.5.5.6.3.cmml">i</mi></msub><mrow id="S4.E2.m1.5.5.5.5" xref="S4.E2.m1.5.5.5.5.cmml"><mrow id="S4.E2.m1.5.5.5.5.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.cmml"><mo id="S4.E2.m1.5.5.5.5.1.1.2" lspace="0em" xref="S4.E2.m1.5.5.5.5.1.1.1.cmml">(</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.cmml"><mrow id="S4.E2.m1.5.5.5.5.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.1.2.cmml"><mi id="S4.E2.m1.1.1.1.1" xref="S4.E2.m1.1.1.1.1.cmml">log</mi><mo id="S4.E2.m1.5.5.5.5.1.1.1.1.1a" xref="S4.E2.m1.5.5.5.5.1.1.1.1.2.cmml">â¡</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.1.2.cmml"><mo id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.2" stretchy="false" xref="S4.E2.m1.5.5.5.5.1.1.1.1.2.cmml">(</mo><msub id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.cmml"><mi id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.2" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.2.cmml">h</mi><mi id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.3" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.3.cmml">i</mi></msub><mo id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.3" stretchy="false" xref="S4.E2.m1.5.5.5.5.1.1.1.1.2.cmml">)</mo></mrow></mrow><mo id="S4.E2.m1.5.5.5.5.1.1.1.3" xref="S4.E2.m1.5.5.5.5.1.1.1.3.cmml">âˆ’</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.2.cmml"><mi id="S4.E2.m1.4.4.4.4" xref="S4.E2.m1.4.4.4.4.cmml">log</mi><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1a" xref="S4.E2.m1.5.5.5.5.1.1.1.2.2.cmml">â¡</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.2.cmml"><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.2" xref="S4.E2.m1.5.5.5.5.1.1.1.2.2.cmml">(</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.cmml"><msub id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.cmml"><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.2" lspace="0em" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.2.cmml">âˆ‘</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.cmml"><mi id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.2" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.2.cmml">j</mi><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.1.cmml">âˆˆ</mo><msub id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.cmml"><mi id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.2" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.2.cmml">R</mi><mi id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.3" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.3.cmml">i</mi></msub></mrow></msub><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.2.cmml"><mi id="S4.E2.m1.3.3.3.3" xref="S4.E2.m1.3.3.3.3.cmml">exp</mi><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1a" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.2.cmml">â¡</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.2.cmml"><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.2" stretchy="false" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.2.cmml">(</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.2.cmml"><mi id="S4.E2.m1.2.2.2.2" xref="S4.E2.m1.2.2.2.2.cmml">log</mi><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1a" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.2.cmml">â¡</mo><mrow id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.2.cmml"><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.2" stretchy="false" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.2.cmml">(</mo><msub id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.cmml"><mi id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.2" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.2.cmml">h</mi><mi id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.3" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.3.cmml">j</mi></msub><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.3" stretchy="false" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.2.cmml">)</mo></mrow></mrow><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.3" stretchy="false" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.2.cmml">)</mo></mrow></mrow></mrow><mo id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.3" xref="S4.E2.m1.5.5.5.5.1.1.1.2.2.cmml">)</mo></mrow></mrow></mrow><mo id="S4.E2.m1.5.5.5.5.1.1.3" rspace="0.055em" xref="S4.E2.m1.5.5.5.5.1.1.1.cmml">)</mo></mrow><mo id="S4.E2.m1.5.5.5.5.2" rspace="0.222em" xref="S4.E2.m1.5.5.5.5.2.cmml">â‹…</mo><msub id="S4.E2.m1.5.5.5.5.3" xref="S4.E2.m1.5.5.5.5.3.cmml"><mi id="S4.E2.m1.5.5.5.5.3.2" xref="S4.E2.m1.5.5.5.5.3.2.cmml">d</mi><mi id="S4.E2.m1.5.5.5.5.3.3" xref="S4.E2.m1.5.5.5.5.3.3.cmml">i</mi></msub></mrow></mrow><mrow id="S4.E2.m1.5.5.7" xref="S4.E2.m1.5.5.7.cmml"><msub id="S4.E2.m1.5.5.7.1" xref="S4.E2.m1.5.5.7.1.cmml"><mo id="S4.E2.m1.5.5.7.1.2" xref="S4.E2.m1.5.5.7.1.2.cmml">âˆ‘</mo><mi id="S4.E2.m1.5.5.7.1.3" xref="S4.E2.m1.5.5.7.1.3.cmml">i</mi></msub><msub id="S4.E2.m1.5.5.7.2" xref="S4.E2.m1.5.5.7.2.cmml"><mi id="S4.E2.m1.5.5.7.2.2" xref="S4.E2.m1.5.5.7.2.2.cmml">d</mi><mi id="S4.E2.m1.5.5.7.2.3" xref="S4.E2.m1.5.5.7.2.3.cmml">i</mi></msub></mrow></mfrac></mrow></mrow><annotation-xml encoding="MathML-Content" id="S4.E2.m1.5b"><apply id="S4.E2.m1.5.6.cmml" xref="S4.E2.m1.5.6"><eq id="S4.E2.m1.5.6.1.cmml" xref="S4.E2.m1.5.6.1"></eq><ci id="S4.E2.m1.5.6.2a.cmml" xref="S4.E2.m1.5.6.2"><mtext id="S4.E2.m1.5.6.2.cmml" xref="S4.E2.m1.5.6.2">Loss</mtext></ci><apply id="S4.E2.m1.5.6.3.cmml" xref="S4.E2.m1.5.6.3"><minus id="S4.E2.m1.5.6.3.1.cmml" xref="S4.E2.m1.5.6.3"></minus><apply id="S4.E2.m1.5.5.cmml" xref="S4.E2.m1.5.5"><divide id="S4.E2.m1.5.5.6.cmml" xref="S4.E2.m1.5.5"></divide><apply id="S4.E2.m1.5.5.5.cmml" xref="S4.E2.m1.5.5.5"><apply id="S4.E2.m1.5.5.5.6.cmml" xref="S4.E2.m1.5.5.5.6"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.5.6.1.cmml" xref="S4.E2.m1.5.5.5.6">subscript</csymbol><sum id="S4.E2.m1.5.5.5.6.2.cmml" xref="S4.E2.m1.5.5.5.6.2"></sum><ci id="S4.E2.m1.5.5.5.6.3.cmml" xref="S4.E2.m1.5.5.5.6.3">ğ‘–</ci></apply><apply id="S4.E2.m1.5.5.5.5.cmml" xref="S4.E2.m1.5.5.5.5"><ci id="S4.E2.m1.5.5.5.5.2.cmml" xref="S4.E2.m1.5.5.5.5.2">â‹…</ci><apply id="S4.E2.m1.5.5.5.5.1.1.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1"><minus id="S4.E2.m1.5.5.5.5.1.1.1.3.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.3"></minus><apply id="S4.E2.m1.5.5.5.5.1.1.1.1.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1"><log id="S4.E2.m1.1.1.1.1.cmml" xref="S4.E2.m1.1.1.1.1"></log><apply id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1">subscript</csymbol><ci id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.2">â„</ci><ci id="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.3.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.1.1.1.1.3">ğ‘–</ci></apply></apply><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1"><log id="S4.E2.m1.4.4.4.4.cmml" xref="S4.E2.m1.4.4.4.4"></log><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1"><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2">subscript</csymbol><sum id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.2"></sum><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3"><in id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.1"></in><ci id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.2">ğ‘—</ci><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3">subscript</csymbol><ci id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.2">ğ‘…</ci><ci id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.3.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.2.3.3.3">ğ‘–</ci></apply></apply></apply><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1"><exp id="S4.E2.m1.3.3.3.3.cmml" xref="S4.E2.m1.3.3.3.3"></exp><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1"><log id="S4.E2.m1.2.2.2.2.cmml" xref="S4.E2.m1.2.2.2.2"></log><apply id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.1.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1">subscript</csymbol><ci id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.2.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.2">â„</ci><ci id="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.3.cmml" xref="S4.E2.m1.5.5.5.5.1.1.1.2.1.1.1.1.1.1.1.1.1.1.3">ğ‘—</ci></apply></apply></apply></apply></apply></apply><apply id="S4.E2.m1.5.5.5.5.3.cmml" xref="S4.E2.m1.5.5.5.5.3"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.5.5.3.1.cmml" xref="S4.E2.m1.5.5.5.5.3">subscript</csymbol><ci id="S4.E2.m1.5.5.5.5.3.2.cmml" xref="S4.E2.m1.5.5.5.5.3.2">ğ‘‘</ci><ci id="S4.E2.m1.5.5.5.5.3.3.cmml" xref="S4.E2.m1.5.5.5.5.3.3">ğ‘–</ci></apply></apply></apply><apply id="S4.E2.m1.5.5.7.cmml" xref="S4.E2.m1.5.5.7"><apply id="S4.E2.m1.5.5.7.1.cmml" xref="S4.E2.m1.5.5.7.1"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.7.1.1.cmml" xref="S4.E2.m1.5.5.7.1">subscript</csymbol><sum id="S4.E2.m1.5.5.7.1.2.cmml" xref="S4.E2.m1.5.5.7.1.2"></sum><ci id="S4.E2.m1.5.5.7.1.3.cmml" xref="S4.E2.m1.5.5.7.1.3">ğ‘–</ci></apply><apply id="S4.E2.m1.5.5.7.2.cmml" xref="S4.E2.m1.5.5.7.2"><csymbol cd="ambiguous" id="S4.E2.m1.5.5.7.2.1.cmml" xref="S4.E2.m1.5.5.7.2">subscript</csymbol><ci id="S4.E2.m1.5.5.7.2.2.cmml" xref="S4.E2.m1.5.5.7.2.2">ğ‘‘</ci><ci id="S4.E2.m1.5.5.7.2.3.cmml" xref="S4.E2.m1.5.5.7.2.3">ğ‘–</ci></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.E2.m1.5c">\text{Loss}=-\frac{\sum_{i}\left(\log(h_{i})-\log\left(\sum_{j\in R_{i}}\exp(%
\log(h_{j}))\right)\right)\cdot d_{i}}{\sum_{i}d_{i}}</annotation><annotation encoding="application/x-llamapun" id="S4.E2.m1.5d">Loss = - divide start_ARG âˆ‘ start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT ( roman_log ( italic_h start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT ) - roman_log ( âˆ‘ start_POSTSUBSCRIPT italic_j âˆˆ italic_R start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT end_POSTSUBSCRIPT roman_exp ( roman_log ( italic_h start_POSTSUBSCRIPT italic_j end_POSTSUBSCRIPT ) ) ) ) â‹… italic_d start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT end_ARG start_ARG âˆ‘ start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT italic_d start_POSTSUBSCRIPT italic_i end_POSTSUBSCRIPT end_ARG</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right" rowspan="1"><span class="ltx_tag ltx_tag_equation ltx_align_right">(2)</span></td>
</tr></tbody>
</table>
<p class="ltx_p" id="S4.SS4.p1.3">where <math alttext="h=\exp(\log(h))" class="ltx_Math" display="inline" id="S4.SS4.p1.1.m1.4"><semantics id="S4.SS4.p1.1.m1.4a"><mrow id="S4.SS4.p1.1.m1.4.4" xref="S4.SS4.p1.1.m1.4.4.cmml"><mi id="S4.SS4.p1.1.m1.4.4.3" xref="S4.SS4.p1.1.m1.4.4.3.cmml">h</mi><mo id="S4.SS4.p1.1.m1.4.4.2" xref="S4.SS4.p1.1.m1.4.4.2.cmml">=</mo><mrow id="S4.SS4.p1.1.m1.4.4.1.1" xref="S4.SS4.p1.1.m1.4.4.1.2.cmml"><mi id="S4.SS4.p1.1.m1.3.3" xref="S4.SS4.p1.1.m1.3.3.cmml">exp</mi><mo id="S4.SS4.p1.1.m1.4.4.1.1a" xref="S4.SS4.p1.1.m1.4.4.1.2.cmml">â¡</mo><mrow id="S4.SS4.p1.1.m1.4.4.1.1.1" xref="S4.SS4.p1.1.m1.4.4.1.2.cmml"><mo id="S4.SS4.p1.1.m1.4.4.1.1.1.2" stretchy="false" xref="S4.SS4.p1.1.m1.4.4.1.2.cmml">(</mo><mrow id="S4.SS4.p1.1.m1.4.4.1.1.1.1.2" xref="S4.SS4.p1.1.m1.4.4.1.1.1.1.1.cmml"><mi id="S4.SS4.p1.1.m1.1.1" xref="S4.SS4.p1.1.m1.1.1.cmml">log</mi><mo id="S4.SS4.p1.1.m1.4.4.1.1.1.1.2a" xref="S4.SS4.p1.1.m1.4.4.1.1.1.1.1.cmml">â¡</mo><mrow id="S4.SS4.p1.1.m1.4.4.1.1.1.1.2.1" xref="S4.SS4.p1.1.m1.4.4.1.1.1.1.1.cmml"><mo id="S4.SS4.p1.1.m1.4.4.1.1.1.1.2.1.1" stretchy="false" xref="S4.SS4.p1.1.m1.4.4.1.1.1.1.1.cmml">(</mo><mi id="S4.SS4.p1.1.m1.2.2" xref="S4.SS4.p1.1.m1.2.2.cmml">h</mi><mo id="S4.SS4.p1.1.m1.4.4.1.1.1.1.2.1.2" stretchy="false" xref="S4.SS4.p1.1.m1.4.4.1.1.1.1.1.cmml">)</mo></mrow></mrow><mo id="S4.SS4.p1.1.m1.4.4.1.1.1.3" stretchy="false" xref="S4.SS4.p1.1.m1.4.4.1.2.cmml">)</mo></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S4.SS4.p1.1.m1.4b"><apply id="S4.SS4.p1.1.m1.4.4.cmml" xref="S4.SS4.p1.1.m1.4.4"><eq id="S4.SS4.p1.1.m1.4.4.2.cmml" xref="S4.SS4.p1.1.m1.4.4.2"></eq><ci id="S4.SS4.p1.1.m1.4.4.3.cmml" xref="S4.SS4.p1.1.m1.4.4.3">â„</ci><apply id="S4.SS4.p1.1.m1.4.4.1.2.cmml" xref="S4.SS4.p1.1.m1.4.4.1.1"><exp id="S4.SS4.p1.1.m1.3.3.cmml" xref="S4.SS4.p1.1.m1.3.3"></exp><apply id="S4.SS4.p1.1.m1.4.4.1.1.1.1.1.cmml" xref="S4.SS4.p1.1.m1.4.4.1.1.1.1.2"><log id="S4.SS4.p1.1.m1.1.1.cmml" xref="S4.SS4.p1.1.m1.1.1"></log><ci id="S4.SS4.p1.1.m1.2.2.cmml" xref="S4.SS4.p1.1.m1.2.2">â„</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S4.SS4.p1.1.m1.4c">h=\exp(\log(h))</annotation><annotation encoding="application/x-llamapun" id="S4.SS4.p1.1.m1.4d">italic_h = roman_exp ( roman_log ( italic_h ) )</annotation></semantics></math> are the hazards, <math alttext="R" class="ltx_Math" display="inline" id="S4.SS4.p1.2.m2.1"><semantics id="S4.SS4.p1.2.m2.1a"><mi id="S4.SS4.p1.2.m2.1.1" xref="S4.SS4.p1.2.m2.1.1.cmml">R</mi><annotation-xml encoding="MathML-Content" id="S4.SS4.p1.2.m2.1b"><ci id="S4.SS4.p1.2.m2.1.1.cmml" xref="S4.SS4.p1.2.m2.1.1">ğ‘…</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.SS4.p1.2.m2.1c">R</annotation><annotation encoding="application/x-llamapun" id="S4.SS4.p1.2.m2.1d">italic_R</annotation></semantics></math> is the risk set, and <math alttext="d" class="ltx_Math" display="inline" id="S4.SS4.p1.3.m3.1"><semantics id="S4.SS4.p1.3.m3.1a"><mi id="S4.SS4.p1.3.m3.1.1" xref="S4.SS4.p1.3.m3.1.1.cmml">d</mi><annotation-xml encoding="MathML-Content" id="S4.SS4.p1.3.m3.1b"><ci id="S4.SS4.p1.3.m3.1.1.cmml" xref="S4.SS4.p1.3.m3.1.1">ğ‘‘</ci></annotation-xml><annotation encoding="application/x-tex" id="S4.SS4.p1.3.m3.1c">d</annotation><annotation encoding="application/x-llamapun" id="S4.SS4.p1.3.m3.1d">italic_d</annotation></semantics></math> represents the event. The optimisation, subject to early stopping criteria, ranged from 20 to 50 minutes on an NVIDIA Tesla V100 32GB GPU. The best model weights, based on the minimal NLL loss (<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.E2" title="2 â€£ 4.4 MM-SurvNet Model Training â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">2</span></a>) were retained for testing.</p>
</div>
<figure class="ltx_figure" id="S4.F5"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="192" id="S4.F5.g1" src="extracted/5416105/figure/survival_curves_10_years.png" width="287"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>Mean survival curves aggregated from all five cross-validation (CV) tests with 95% confidence intervals. In the depicted curves, the log-rank test p<math alttext="&lt;" class="ltx_Math" display="inline" id="S4.F5.2.m1.1"><semantics id="S4.F5.2.m1.1b"><mo id="S4.F5.2.m1.1.1" xref="S4.F5.2.m1.1.1.cmml">&lt;</mo><annotation-xml encoding="MathML-Content" id="S4.F5.2.m1.1c"><lt id="S4.F5.2.m1.1.1.cmml" xref="S4.F5.2.m1.1.1"></lt></annotation-xml><annotation encoding="application/x-tex" id="S4.F5.2.m1.1d">&lt;</annotation><annotation encoding="application/x-llamapun" id="S4.F5.2.m1.1e">&lt;</annotation></semantics></math>0.05.</figcaption>
</figure>
</section>
</section>
<section class="ltx_section" id="S5">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Experimental Results</h2>
<div class="ltx_para ltx_noindent" id="S5.p1">
<p class="ltx_p" id="S5.p1.1">In our study on survival risk stratification for ER+ breast cancer patients, we integrated multimodal dataâ€”histopathological images, genetic information, and clinical dataâ€”resulting in enhanced predictive accuracy. Our method achieved a notable C-index of 0.64, superior to models leveraging a single data modality (Table <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.T1" title="Table 1 â€£ 4.2 Patch-to-Patient Aggregation Using Self-Attention â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">1</span></a>). For instance, models based on histopathological image data only or on clinical information only, yielded C-indices of 0.53 and 0.47, respectively. An ablation study (TableÂ <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.T1" title="Table 1 â€£ 4.2 Patch-to-Patient Aggregation Using Self-Attention â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">1</span></a>) highlighted the significance of our holistic, multimodal approach. When integrating all feature sets, the MaxViT model surpassed ResNet50, emphasising the advantages of combining clinical, imaging and genetic data for more nuanced and accurate risk predictions. Kaplan-Meier survival analysis (Fig.Â <a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#S4.F5" title="Figure 5 â€£ 4.4 MM-SurvNet Model Training â€£ 4 Proposed Method â€£ MM-SurvNet: Deep Learning-Based Survival Risk Stratification in Breast Cancer Through Multimodal Data Fusion"><span class="ltx_text ltx_ref_tag">5</span></a>) further demonstrates our methodâ€™s efficacy in differentiating outcomes between risk groups (log-rank test p<math alttext="&lt;" class="ltx_Math" display="inline" id="S5.p1.1.m1.1"><semantics id="S5.p1.1.m1.1a"><mo id="S5.p1.1.m1.1.1" xref="S5.p1.1.m1.1.1.cmml">&lt;</mo><annotation-xml encoding="MathML-Content" id="S5.p1.1.m1.1b"><lt id="S5.p1.1.m1.1.1.cmml" xref="S5.p1.1.m1.1.1"></lt></annotation-xml><annotation encoding="application/x-tex" id="S5.p1.1.m1.1c">&lt;</annotation><annotation encoding="application/x-llamapun" id="S5.p1.1.m1.1d">&lt;</annotation></semantics></math>0.05), with an Integrated Brier Score of 0.11 for 5 years and 0.16 for 10 yearsÂ <cite class="ltx_cite ltx_citemacro_cite">[<a class="ltx_ref" href="https://arxiv.org/html/2402.11788v1#bib.bib27" title="">27</a>]</cite>.</p>
</div>
</section>
<section class="ltx_section" id="S6">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Discussion</h2>
<div class="ltx_para ltx_noindent" id="S6.p1">
<p class="ltx_p" id="S6.p1.1">The findings of our study assessing multimodal data integration for survival risk stratification in ER+ breast cancer patients underscore the superiority of multimodal approaches. Our proposed model, amalgamating histopathological images, genetic data and clinical parameters, achieved a significant C-index of 0.64, outshining single-modal counterparts. Specifically, the MaxViT architecture consistently surpassed ResNet50 in handling this multimodal data. While the Kaplan-Meier analysis reinforced our modelâ€™s aptitude in patient risk stratification, there was notable performance variability across cross-validation folds, suggesting potential dataset nuances or imbalances. Moreover, the traditional CoxPH modelâ€™s C-index of 0.47 highlighted the advantages of deep learning in capturing cancer prognosisâ€™ intricate nature. The analysis not only substantiated our modelâ€™s proficiency in stratifying patients but also highlighted significant survival differences between the high-risk and low-risk groups, as revealed in the survival curve plots.</p>
</div>
<div class="ltx_para ltx_noindent" id="S6.p2">
<p class="ltx_p" id="S6.p2.1">Future work will encompass broader data types and probe the modelâ€™s utility across diverse cancer subtypes. While our study yielded promising results, it suffers from certain limitations. Our sample size, though adequate for the current modeling exercise, might benefit from expansion to ensure broader generalisability. The studyâ€™s concentration on ER+ breast cancer means that its applicability to other cancer subtypes remains as yet untested. The variability in performance across different cross-validation folds suggests potential dataset imbalances or unidentified influencing factors. Lastly, the modelâ€™s dependence on three types of data may pose challenges in settings where one or more of these data modalities is unavailable.</p>
</div>
</section>
<section class="ltx_section" id="S7">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">7 </span>Conclusion</h2>
<div class="ltx_para ltx_noindent" id="S7.p1">
<p class="ltx_p" id="S7.p1.1">Our study highlights the significance of multimodal data integration in advancing survival risk stratification for ER+ breast cancer patients. This integrative methodology enhances prediction accuracy and sets the foundation for personalised treatment strategies in oncology.</p>
</div>
</section>
<section class="ltx_section" id="Sx1">
<h2 class="ltx_title ltx_title_section">Compliance with Ethical Standards</h2>
<div class="ltx_para ltx_noindent" id="Sx1.p1">
<p class="ltx_p" id="Sx1.p1.1">This research study was conducted retrospectively using human subject data made available in open access by The Cancer Genome Atlas Breast Cancer (TCGA-BRCA) dataset, accessible through the National Cancer Instituteâ€™s <a class="ltx_ref ltx_href" href="https://portal.gdc.cancer.gov/projects/TCGA-BRCA" title="">Genomic Data Commons (GDC)</a> portal. Ethical approval was not required for this study, in accordance with the <a class="ltx_ref ltx_href" href="https://www.cancer.gov/ccg/research/genome-sequencing/tcga/history/ethics-policies" title="">ethical policies</a> set forth by The Cancer Genome Atlas program.</p>
</div>
</section>
<section class="ltx_section" id="Sx2">
<h2 class="ltx_title ltx_title_section">Conflicts of Interest</h2>
<div class="ltx_para ltx_noindent" id="Sx2.p1">
<p class="ltx_p" id="Sx2.p1.1">No funding was received for conducting this study. The authors have no relevant financial or non-financial interests to disclose.</p>
</div>
</section>
<section class="ltx_section" id="Sx3">
<h2 class="ltx_title ltx_title_section">Acknowledgement</h2>
<div class="ltx_para" id="Sx3.p1">
<p class="ltx_p" id="Sx3.p1.1">This research was undertaken with the assistance of resources and services from the National Computational Infrastructure (NCI), which is supported by the Australian Government. Additionally, data preprocessing was performed using the computational cluster Katana, which is supported by Research Technology Services at UNSW Sydney.
</p>
</div>
</section>
<section class="ltx_bibliography" id="bib">
<h2 class="ltx_title ltx_title_bibliography">References</h2>
<ul class="ltx_biblist">
<li class="ltx_bibitem" id="bib.bib1">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[1]</span>
<span class="ltx_bibblock">
H.Â Lindman, F.Â Wiklund, and K.Â K. Andersen, â€œLong-term treatment patterns and survival in metastatic breast cancer by intrinsic subtypesâ€”an observational cohort study in Sweden,â€ <span class="ltx_text ltx_font_italic" id="bib.bib1.1.1">BMC Cancer</span>, vol.Â 22, p.Â 1006, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib2">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[2]</span>
<span class="ltx_bibblock">
Y.Â Han, J.Â Wang, and B.Â Xu, â€œClinicopathological characteristics and prognosis of breast cancer with special histological types: A surveillance, epidemiology, and end results database analysis,â€ <span class="ltx_text ltx_font_italic" id="bib.bib2.1.1">The Breast</span>, vol.Â 54, pp.Â 114â€“120, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib3">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[3]</span>
<span class="ltx_bibblock">
C.Â Shuai, F.Â Yuan, Y.Â Liu, C.Â Wang, J.Â Wang, and H.Â He, â€œEstrogen receptorâ€”positive breast cancer survival prediction and analysis of resistanceâ€“related genes introduction,â€ <span class="ltx_text ltx_font_italic" id="bib.bib3.1.1">PeerJ</span>, vol.Â 9, p.Â e12202, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib4">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[4]</span>
<span class="ltx_bibblock">
X.Â Li, L.Â Liu, G.Â J. Goodall, A.Â W. Schreiber, T.Â Xu, J.Â Li, and T.Â D. Le, â€œA novel single-cell based method for breast cancer prognosis,â€ <span class="ltx_text ltx_font_italic" id="bib.bib4.1.1">PLoS Computational Biology</span>, vol.Â 16, p.Â e1008133, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib5">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[5]</span>
<span class="ltx_bibblock">
V.Â Subramanian, T.Â Syeda-Mahmood, and M.Â N. Do, â€œMulti-modality fusion using canonical correlation analysis methods: Application in breast cancer survival prediction from histology and genomics,â€ 11 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib6">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[6]</span>
<span class="ltx_bibblock">
C.Â Nero, F.Â Ciccarone, A.Â Pietragalla, S.Â Duranti, G.Â Daniele, G.Â Scambia, and D.Â Lorusso, â€œAdjuvant treatment recommendations in early-stage endometrial cancer: What changes with the introduction of the integrated molecular-based risk assessment,â€ <span class="ltx_text ltx_font_italic" id="bib.bib6.1.1">Frontiers in Oncology</span>, vol.Â 11, p.Â 612450, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib7">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[7]</span>
<span class="ltx_bibblock">
W.Â Guo, W.Â Liang, Q.Â Deng, and X.Â Zou, â€œA multimodal affinity fusion network for predicting the survival of breast cancer patients,â€ <span class="ltx_text ltx_font_italic" id="bib.bib7.1.1">Frontiers in Genetics</span>, vol.Â 12, p.Â 709027, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib8">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[8]</span>
<span class="ltx_bibblock">
Y.Â B. Shvetsov, L.Â R. Wilkens, K.Â K. White, M.Â Chong, A.Â Buyum, G.Â Badowski, R.Â T.Â L. Guerrero, and R.Â Novotny, â€œPrediction of breast cancer risk among women of the Mariana Islands: The BRISK retrospective caseâ€”control study,â€ <span class="ltx_text ltx_font_italic" id="bib.bib8.1.1">BMJ Open</span>, vol.Â 12, p.Â e061205, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib9">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[9]</span>
<span class="ltx_bibblock">
K.Â Holli-Helenius, A.Â Salminen, I.Â Rinta-Kiikka, I.Â Koskivuo, N.Â BrÃ¼ck, P.Â BostrÃ¶m, and R.Â Parkkola, â€œMRI texture analysis in differentiating luminal A and luminal B breast cancer molecular subtypesâ€”a feasibility study,â€ <span class="ltx_text ltx_font_italic" id="bib.bib9.1.1">BMC Medical Imaging</span>, vol.Â 17, p.Â 69, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib10">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[10]</span>
<span class="ltx_bibblock">
K.Â Yao, E.Â Schaafsma, B.Â Zhang, and C.Â Cheng, â€œTumor cell intrinsic and extrinsic features predict prognosis in estrogen receptor positive breast cancer,â€ <span class="ltx_text ltx_font_italic" id="bib.bib10.1.1">PLOS Computational Biology</span>, vol.Â 18, pp.Â 1â€“22, 03 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib11">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[11]</span>
<span class="ltx_bibblock">
S.Â C. Wetstein, V.Â M.Â d. Jong, N.Â Stathonikos, M.Â Opdam, G.Â M. H.Â E. Dackus, J.Â P.Â W. Pluim, P.Â J.Â v. Diest, and M.Â Veta, â€œDeep learning-based breast cancer grading and survival analysis on whole-slide histopathology images,â€ <span class="ltx_text ltx_font_italic" id="bib.bib11.1.1">Scientific Reports</span>, vol.Â 12, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib12">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[12]</span>
<span class="ltx_bibblock">
T.Â Wei, X.Â Yuan, R.Â Gao, L.Â J. Johnston, J.Â Zhou, Y.Â Wang, W.Â Kong, Y.Â Xie, Y.Â Zhang, D.Â Xu, and Z.Â Yu, â€œSurvival prediction of stomach cancer using expression data and deep learning models with histopathological images,â€ <span class="ltx_text ltx_font_italic" id="bib.bib12.1.1">Cancer Science</span>, vol.Â 114, pp.Â 690â€“701, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib13">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[13]</span>
<span class="ltx_bibblock">
L.Â Chen, H.Â Zeng, X.Â Yu, Y.Â Huang, Y.Â Luo, and X.Â Ma, â€œHistopathological images and multi-omics integration predict molecular characteristics and survival in lung adenocarcinoma,â€ <span class="ltx_text ltx_font_italic" id="bib.bib13.1.1">Frontiers in Cell and Developmental Biology</span>, vol.Â 9, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib14">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[14]</span>
<span class="ltx_bibblock">
X.Â Wu, Y.Â Shi, M.Â Wang, and A.Â Li, â€œCAMR: cross-aligned multimodal representation learning for cancer survival prediction,â€ <span class="ltx_text ltx_font_italic" id="bib.bib14.1.1">Bioinformatics</span>, vol.Â 39, p.Â btad025, 2023.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib15">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[15]</span>
<span class="ltx_bibblock">
Y.Â He, B.Â Hu, C.Â Zhu, W.Â Xu, X.Â Hao, B.Â Dong, X.Â Chen, Q.Â Dong, and X.Â Zhou, â€œA novel multimodal radiomics model for predicting prognosis of resected hepatocellular carcinoma,â€ <span class="ltx_text ltx_font_italic" id="bib.bib15.1.1">Frontiers in Oncology</span>, vol.Â 12, p.Â 745258, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib16">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[16]</span>
<span class="ltx_bibblock">
V.Â Subramanian, T.Â Syeda-Mahmood, and M.Â N. Do, â€œMultimodal fusion using sparse CCA for breast cancer survival prediction,â€ <span class="ltx_text ltx_font_italic" id="bib.bib16.1.1">IEEE International Symposium on Biomedical Imaging</span>, pp.Â 1429â€“1432, 2021.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib17">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[17]</span>
<span class="ltx_bibblock">
R.Â Vanguri, J.Â Luo, A.Â Aukerman, J.Â V. Egger, C.Â J. Fong, N.Â Horvat, A.Â Pagano, J.Â d. A.Â B. AraÃºjo-Filho, L.Â Geneslaw, H.Â Rizvi, R.Â E. Sosa, K.Â M. Boehm, S.Â Yang, F.Â M. Bodd, K.Â Ventura, T.Â J. Hollmann, M.Â S. Ginsberg, J.Â Gao, M.Â D. Hellmann, J.Â L. Sauter, and S.Â P. Shah, â€œMultimodal integration of radiology, pathology and genomics for prediction of response to PD-(L)1 blockade in patients with non-small cell lung cancer,â€ <span class="ltx_text ltx_font_italic" id="bib.bib17.1.1">Nature Cancer</span>, vol.Â 3, no.Â 10, pp.Â 1151â€“1164, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib18">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[18]</span>
<span class="ltx_bibblock">
W.Â Lingle, B.Â J. Erickson, M.Â L. Zuley, R.Â Jarosz, E.Â Bonaccio, J.Â Filippini, J.Â M. Net, L.Â Levi, E.Â A. Morris, G.Â G. Figler, P.Â Elnajjar, S.Â Kirk, Y.Â Lee, M.Â Giger, and N.Â Gruszauskas, â€œThe cancer genome atlas breast invasive carcinoma collection (TCGA-BRCA),â€ <span class="ltx_text ltx_font_italic" id="bib.bib18.1.1">The Cancer Imaging Archive</span>, 2016.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib19">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[19]</span>
<span class="ltx_bibblock">
P.Â Bankhead, M.Â B. Loughrey, J.Â A. FernÃ¡ndez, Y.Â Dombrowski, D.Â G. McArt, P.Â D. Dunne, S.Â McQuaid, R.Â T. Gray, L.Â J. Murray, H.Â G. Coleman, J.Â A. James, M.Â Salto-Tellez, and P.Â W. Hamilton, â€œQuPath: Open source software for digital pathology image analysis,â€ <span class="ltx_text ltx_font_italic" id="bib.bib19.1.1">Scientific Reports</span>, vol.Â 7, p.Â 16878, 2017.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib20">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[20]</span>
<span class="ltx_bibblock">
M.Â Macenko, M.Â Niethammer, J.Â S. Marron, D.Â Borland, J.Â T. Woosley, X.Â Guan, C.Â Schmitt, and N.Â E. Thomas, â€œA method for normalizing histology slides for quantitative analysis,â€ <span class="ltx_text ltx_font_italic" id="bib.bib20.1.1">IEEE International Symposium on Biomedical Imaging (ISBI)</span>, pp.Â 1107â€“1110, 2009.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib21">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[21]</span>
<span class="ltx_bibblock">
B.Â Li and C.Â N. Dewey, â€œRSEM: Accurate transcript quantification from RNA-Seq data with or without a reference genome,â€ <span class="ltx_text ltx_font_italic" id="bib.bib21.1.1">BMC Bioinformatics</span>, vol.Â 12, p.Â 323, 2011.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib22">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[22]</span>
<span class="ltx_bibblock">
J.Â S. Parker, M.Â E. Mullins, M.Â C. Cheang, S.Â Leung, V.Â David, T.Â L. Vickery, S.Â R. Davies, C.Â Fauron, X.Â He, Z.Â Hu, J.Â Quackenbush, I.Â J. Stijleman, J.Â Palazzo, J.Â S. Marron, A.Â B. Nobel, E.Â R. Mardis, T.Â O. Nielsen, M.Â J. Ellis, C.Â M. Perou, and P.Â S. Bernard, â€œSupervised risk predictor of breast cancer based on intrinsic subtypes,â€ <span class="ltx_text ltx_font_italic" id="bib.bib22.1.1">Journal of Clinical Oncology</span>, vol.Â 27, pp.Â 1160â€“1167, 2009.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib23">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[23]</span>
<span class="ltx_bibblock">
Z.Â Tu, H.Â Talebi, H.Â Zhang, F.Â Yang, P.Â Milanfar, A.Â Bovik, and Y.Â Li, â€œMaxViT: Multi-axis vision transformer,â€ <span class="ltx_text ltx_font_italic" id="bib.bib23.1.1">European Conference on Computer Vision (ECCV)</span>, p.Â 459â€“479, 2022.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib24">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[24]</span>
<span class="ltx_bibblock">
A.Â D. Jones, J.Â P. Graff, M.Â A. Darrow, A.Â D. Borowsky, K.Â Olson, R.Â Gandour-Edwards, A.Â Mitra, D.Â Wei, G.Â Gao, B.Â Durbin-Johnson, and H.Â H. Rashidi, â€œImpact of pre-analytical variables on deep learning accuracy in histopathology,â€ <span class="ltx_text ltx_font_italic" id="bib.bib24.1.1">Histopathology</span>, vol.Â 75, pp.Â 39â€“53, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib25">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[25]</span>
<span class="ltx_bibblock">
I.Â Loshchilov and F.Â Hutter, â€œDecoupled weight decay regularization,â€ <span class="ltx_text ltx_font_italic" id="bib.bib25.1.1">International Conference on Learning Representations (ICLR)</span>, 2019.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib26">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[26]</span>
<span class="ltx_bibblock">
E.Â Longato, M.Â Vettoretti, and B.Â Di Camillo, â€œA practical perspective on the concordance index for the evaluation and selection of prognostic time-to-event models,â€ <span class="ltx_text ltx_font_italic" id="bib.bib26.1.1">Journal of Biomedical Informatics</span>, vol.Â 108, p.Â 103496, 2020.

</span>
</li>
<li class="ltx_bibitem" id="bib.bib27">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">[27]</span>
<span class="ltx_bibblock">
K.Â J. Jager, P.Â C. van Dijk, C.Â Zoccali, and F.Â W. Dekker, â€œThe analysis of survival data: The Kaplanâ€“Meier method,â€ <span class="ltx_text ltx_font_italic" id="bib.bib27.1.1">Kidney International</span>, vol.Â 74, no.Â 5, pp.Â 560â€“565, 2008.

</span>
</li>
</ul>
</section>
<div class="ltx_para ltx_minipage ltx_align_top" id="p4" style="width:216.8pt;">
<div class="ltx_block ltx_parbox ltx_align_top" id="p4.1" style="width:72.3pt;">
<img alt="[Uncaptioned image]" class="ltx_graphics ltx_img_portrait" height="125" id="p4.1.g1" src="extracted/5416105/author_pic/raktim.jpg" width="100"/>
</div>
<p class="ltx_p ltx_minipage ltx_align_top" id="p4.3" style="width:140.9pt;"><a class="ltx_ref ltx_href ltx_font_sansserif" href="https://research.unsw.edu.au/people/mr-raktim-kumar-mondol" style="color:#0000FF;" title="">Raktim Kumar Mondol</a> <span class="ltx_text ltx_font_sansserif" id="p4.3.1" style="color:#000000;">is a PhD candidate in Computer Science and Engineering, specializing in computer vision and bioinformatics. He completed his MEng in Engineering with High Distinction from RMIT University, Australia. Mondolâ€™s research interests include histopathological image analysis, clinical prognosis prediction, and enhancing clinical understanding through the interpretability of computational models.</span></p>
</div>
<div class="ltx_para ltx_minipage ltx_align_top" id="p5" style="width:216.8pt;">
<div class="ltx_block ltx_parbox ltx_align_top" id="p5.1" style="width:72.3pt;">
<img alt="[Uncaptioned image]" class="ltx_graphics ltx_img_portrait" height="125" id="p5.1.g1" src="extracted/5416105/author_pic/ewan.jpeg" width="100"/>
</div>
<p class="ltx_p ltx_minipage ltx_align_top" id="p5.3" style="width:140.9pt;"><a class="ltx_ref ltx_href ltx_font_sansserif" href="https://www.unsw.edu.au/staff/ewan-millar" style="color:#0000FF;" title="">Ewan Millar</a> <span class="ltx_text ltx_font_sansserif" id="p5.3.1" style="color:#000000;">is a Senior Staff Specialist Histopathologist with NSW Health Pathology at St George Hospital Sydney with expertise in breast cancer pathology and translational research and a strong interest in AI and digital pathology applications.</span></p>
</div>
<div class="ltx_para ltx_minipage ltx_align_top" id="p6" style="width:216.8pt;">
<div class="ltx_block ltx_parbox ltx_align_top" id="p6.1" style="width:72.3pt;">
<img alt="[Uncaptioned image]" class="ltx_graphics ltx_img_portrait" height="125" id="p6.1.g1" src="extracted/5416105/author_pic/sowmya.jpg" width="100"/>
</div>
<p class="ltx_p ltx_minipage ltx_align_top" id="p6.3" style="width:140.9pt;"><a class="ltx_ref ltx_href ltx_font_sansserif" href="https://research.unsw.edu.au/people/professor-arcot-sowmya" style="color:#0000FF;" title="">Arcot Sowmya</a> <span class="ltx_text ltx_font_sansserif" id="p6.3.1" style="color:#000000;">is Professor in the School of Computer Science and Engineering, UNSW. Her major research interest is in the area of Machine Learning for Computer Vision and includes learning object models, feature extraction, segmentation and recognition based on computer vision, machine learning and deep learning. In recent years, applications in the broader health area are a focus, including biomedical informatics and rapid diagnostics in the real world. All of these areas have been supported by competitive, industry and government funding.</span></p>
</div>
<div class="ltx_para ltx_minipage ltx_align_top" id="p7" style="width:216.8pt;">
<div class="ltx_block ltx_parbox ltx_align_top" id="p7.1" style="width:72.3pt;">
<img alt="[Uncaptioned image]" class="ltx_graphics ltx_img_portrait" height="125" id="p7.1.g1" src="extracted/5416105/author_pic/erik.jpg" width="100"/>
</div>
<p class="ltx_p ltx_minipage ltx_align_top" id="p7.3" style="width:140.9pt;"><a class="ltx_ref ltx_href ltx_font_sansserif" href="https://imagescience.org/meijering/" style="color:#0000FF;" title="">Erik Meijering</a> <span class="ltx_text ltx_font_sansserif" id="p7.3.1" style="color:#000000;">(Fellow, IEEE), is a Professor of Biomedical Image Computing in the School of Computer Science and Engineering. His research focusses on the development of innovative computer vision and machine learning (in particular deep learning) methods for automated quantitative analysis of biomedical imaging data.</span></p>
</div>
<div class="ltx_pagination ltx_role_newpage"></div>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Mon Feb 19 02:30:53 2024 by <a class="ltx_LaTeXML_logo" href="http://dlmf.nist.gov/LaTeXML/"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span style="font-size:70%;position:relative; bottom:2.2pt;">A</span>T<span style="position:relative; bottom:-0.4ex;">E</span></span><span class="ltx_font_smallcaps">xml</span><img alt="[LOGO]" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg=="/></a>
</div></footer>
</div>
</body>
</html>
